#!/usr/bin/env bash
# Sync specific files from S3 to local directory

src="s3://pipeline-prod-cache-503977275616-ap-southeast-2/byob-icav2/production/analysis/oncoanalyser-wgts-dna/20250407e2ff5344/L2500331_L2500332/"
dest="${HOME}/projects/tidywigits/nogit/oncoanalyser-wgts-dna/20250407e2ff5344/L2500331_L2500332"
aws s3 sync ${src} ${dest} \
    --exclude "*" \
    --include "*.duplicate_freq.tsv" \
    --include "*.amber.baf.pcf" \
    --include "*.amber.baf.tsv.gz" \
    --include "*.amber.contamination.tsv" \
    --include "*.amber.contamination.vcf.gz" \
    --include "*.amber.contamination.vcf.gz.tbi" \
    --include "*.amber.qc" \
    --include "*.amber.homozygousregion.tsv" \
    --include "*.amber.snp.vcf.gz" \
    --include "*.amber.snp.vcf.gz.tbi" \
    --include "*.wgsmetrics" \
    --include "*_chord_prediction.txt" \
    --include "*_chord_signatures.txt" \
    --include "*cobalt.version" \
    --include "*.cobalt.gc.median.tsv" \
    --include "*.cobalt.ratio.pcf" \
    --include "*.cobalt.ratio.tsv.gz" \
    --include "*.cobalt.ratio.median.tsv" \
    --include "*_cup_report.pdf" \
    --include "*.cup.data.csv" \
    --include "*.cup.report.summary.png" \
    --include "*.cuppa.chart.png" \
    --include "*.cuppa.conclusion.txt" \
    --include "*.flagstat" \
    --include "*.gridss.vcf.gz" \
    --include "*.lilac.candidates.coverage.tsv" \
    --include "*.lilac.qc.tsv" \
    --include "*.lilac.tsv" \
    --include "*.linx.germline.breakend.tsv" \
    --include "*.linx.germline.clusters.tsv" \
    --include "*.linx.germline.disruption.tsv" \
    --include "*.linx.germline.driver.catalog.tsv" \
    --include "*.linx.germline.links.tsv" \
    --include "*.linx.germline.svs.tsv" \
    --include "*linx.version" \
    --include "*.linx.breakend.tsv" \
    --include "*.linx.clusters.tsv" \
    --include "*.linx.driver.catalog.tsv" \
    --include "*.linx.drivers.tsv" \
    --include "*.linx.fusion.tsv" \
    --include "*.linx.links.tsv" \
    --include "*.linx.svs.tsv" \
    --include "*.linx.vis_copy_number.tsv" \
    --include "*.linx.vis_fusion.tsv" \
    --include "*.linx.vis_gene_exon.tsv" \
    --include "*.linx.vis_protein_domain.tsv" \
    --include "*.linx.vis_segments.tsv" \
    --include "*.linx.vis_sv_data.tsv" \
    --include "*.purple.cnv.gene.tsv" \
    --include "*.purple.cnv.somatic.tsv" \
    --include "*.purple.driver.catalog.germline.tsv" \
    --include "*.purple.driver.catalog.somatic.tsv" \
    --include "*.purple.germline.deletion.tsv" \
    --include "*.purple.germline.vcf.gz" \
    --include "*.purple.germline.vcf.gz.tbi" \
    --include "*.purple.purity.range.tsv" \
    --include "*.purple.purity.tsv" \
    --include "*.purple.qc" \
    --include "*.purple.segment.tsv" \
    --include "*.purple.somatic.clonality.tsv" \
    --include "*.purple.somatic.hist.tsv" \
    --include "*.purple.somatic.vcf.gz" \
    --include "*.purple.somatic.vcf.gz.tbi" \
    --include "*.purple.sv.germline.vcf.gz" \
    --include "*.purple.sv.germline.vcf.gz.tbi" \
    --include "*.purple.sv.vcf.gz" \
    --include "*.purple.sv.vcf.gz.tbi" \
    --include "*purple.version" \
    --include "*.sage.germline.vcf.gz" \
    --include "*.sage.germline.vcf.gz.tbi" \
    --include "*.sage.bqr.png" \
    --include "*.sage.bqr.tsv" \
    --include "*.sage.exon.medians.tsv" \
    --include "*.sage.gene.coverage.tsv" \
    --include "*.sig.snv_counts.csv" \
    --include "*.sig.allocation.tsv" \
    --include "*.virusbreakend.vcf" \
    --include "*.virusbreakend.vcf.summary.tsv" \
    --include "*.virus.annotated.tsv" \
    --dryrun
