<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Schemas Tidy â€¢ tidywigits</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><script src="schemas_tidy_files/libs/quarto-html/popper.min.js"></script><script src="schemas_tidy_files/libs/quarto-html/tippy.umd.min.js"></script><link href="schemas_tidy_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="schemas_tidy_files/libs/quarto-html/light-border.css" rel="stylesheet">
<script src="schemas_tidy_files/libs/core-js-2.5.3/shim.min.js"></script><script src="schemas_tidy_files/libs/react-18.2.0/react.min.js"></script><script src="schemas_tidy_files/libs/react-18.2.0/react-dom.min.js"></script><script src="schemas_tidy_files/libs/reactwidget-2.0.0/react-tools.js"></script><link href="schemas_tidy_files/libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="schemas_tidy_files/libs/htmlwidgets-1.6.4/htmlwidgets.js"></script><link href="schemas_tidy_files/libs/reactable-0.4.4/reactable.css" rel="stylesheet">
<script src="schemas_tidy_files/libs/reactable-binding-0.4.4/reactable.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Schemas Tidy">
<meta property="og:image" content="https://umccr.github.io/tidywigits/logo.png">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-primary" data-bs-theme="dark" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">tidywigits</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.0.4</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../index.html">Intro</a></li>
<li class="active nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles">
<li><a class="dropdown-item" href="../articles/installation.html">Installation</a></li>
    <li><a class="dropdown-item" href="../articles/schemas_raw.html">Schemas Raw</a></li>
    <li><a class="dropdown-item" href="../articles/schemas_tidy.html">Schemas Tidy</a></li>
    <li><a class="dropdown-item" href="../articles/structure.html">Structure</a></li>
    <li><a class="dropdown-item" href="../articles/developers.html">Developers</a></li>
    <li><a class="dropdown-item" href="../articles/uml.html">UML</a></li>
  </ul>
</li>
<li class="nav-item"><a class="nav-link" href="../articles/NEWS.html">NEWS</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/umccr/tidywigits/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-quarto">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>Schemas Tidy</h1>




      <small class="dont-index">Source: <a href="https://github.com/umccr/tidywigits/blob/main/vignettes/schemas_tidy.qmd" class="external-link"><code>vignettes/schemas_tidy.qmd</code></a></small>
      <div class="d-none name"><code></code></div>
    </div>






    <div class="cell">
<div class="cell-output-display">
<div>
<script>
  function toggleSchema(rowId, versionIndex) {
    var schemaId = 'schema_' + rowId + '_' + versionIndex;
    var buttonId = 'btn_' + rowId + '_' + versionIndex;
    var schemaDiv = document.getElementById(schemaId);
    var button = document.getElementById(buttonId);

    if (schemaDiv && button) {
      if (schemaDiv.style.display === 'none' || schemaDiv.style.display === '') {
        // Show schema, change button to active state
        schemaDiv.style.display = 'block';
        button.style.backgroundColor = '#1565c0';
        button.style.color = 'white';
        button.style.borderColor = '#0d47a1';
        button.style.transform = 'scale(0.98)';
      } else {
        // Hide schema, reset button to normal state
        schemaDiv.style.display = 'none';
        button.style.backgroundColor = '#e3f2fd';
        button.style.color = '#1565c0';
        button.style.borderColor = '#90caf9';
        button.style.transform = 'scale(1)';
      }
    }
}
  </script><div class="reactable html-widget html-fill-item" id="htmlwidget-8b2b7a2964af9ff91e3a" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8b2b7a2964af9ff91e3a">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"n":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73],"tool":["ALIGNMENTS","ALIGNMENTS","AMBER","AMBER","AMBER","AMBER","BAMTOOLS","BAMTOOLS","BAMTOOLS","BAMTOOLS","BAMTOOLS","BAMTOOLS","BAMTOOLS","CHORD","CHORD","COBALT","COBALT","COBALT","COBALT","CUPPA","CUPPA","CUPPA","CUPPA","ESVEE","ESVEE","ESVEE","ESVEE","ESVEE","ESVEE","ESVEE","FLAGSTATS","ISOFOX","ISOFOX","ISOFOX","ISOFOX","ISOFOX","ISOFOX","ISOFOX","ISOFOX","LILAC","LILAC","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","LINX","PURPLE","PURPLE","PURPLE","PURPLE","PURPLE","PURPLE","PURPLE","PURPLE","PURPLE","PURPLE","SAGE","SAGE","SAGE","SIGS","SIGS","VIRUSBREAKEND","VIRUSINTERPRETER"],"tbl":["dupfreq","markdup","bafpcf","contaminationtsv","homozygousregion","qc","wgsmetrics","coverage","flagstats","fraglength","partitionstats","summary","summary_2","prediction","signatures","gcmed","ratiomed","ratiopcf","version","datacsv","feat","predsum","datatsv","prepfraglen","prepdiscstats","prepjunction","assemblephased","assembleassembly","assemblebreakend","assemblealignment","flagstats","summary","genedata","transdata","altsj","retintron","fusionsall","fusionspass","genecollection","summary","qc","breakends","clusters","drivercatalog","drivers","fusions","links","neoepitope","svs","viscn","visfusion","visgeneexon","visproteindomain","vissegments","vissvdata","version","cnvgenetsv","cnvsomtsv","drivercatalog","germdeltsv","purityrange","puritytsv","qc","somclonality","somhist","version","bqrtsv","genecvg","exoncvg","allocation","snvcounts","summary","annotated"],"schema_version":[{"version":["latest"],"schema":[{"field":["duplicate_read_count","frequency"],"type":["c","d"],"description":["Duplicate read count","Frequency of duplicate reads"]}]},{"version":["latest"],"schema":[{"field":["library","unpaired_reads_examined","read_pairs_examined","secondary_or_supplementary_reads","unmapped_reads","unpaired_read_duplicates","read_pair_duplicates","read_pair_optical_duplicates","percent_duplication","estimated_library_size","SPLIT_HERE","bin","cov_mult","all_sets","optical_sets","non_optical_sets"],"type":["c","d","d","d","d","d","d","d","d","d","c","d","d","d","d","d"],"description":["library on which the duplicate marking was performed","number of mapped reads examined which did not have a mapped mate pair","number of mapped read pairs examined","number of reads that were either secondary or supplementary","total number of unmapped reads examined","number of fragments that were marked as duplicates","number of read pairs that were marked as duplicates","number of read pair duplicates that were caused by optical duplication","fraction of mapped sequence that is marked as duplicate","estimated number of unique molecules in the library based on PE duplication","internal use for splitting the schema.","coverage bin (maybe?)","coverage multiple","all sets","optical sets","non optical sets"]}]},{"version":["latest"],"schema":[{"field":["sampleid","chrom","arm","start_pos","end_pos","n_probes","mean"],"type":["c","c","c","i","i","i","d"],"description":["sample ID","chromosome","chromosome arm","start position","end position","number of probes","mean BAF value"]}]},{"version":["latest"],"schema":[{"field":["chrom","pos","ref","alt","dp_normal","refsup_normal","altsup_normal","dp_tumor","refsup_tumor","altsup_tumor"],"type":["c","i","c","c","i","i","i","i","i","i"],"description":["chromosome","position of site","reference allele","alternate allele","normal depth","normal reference support","normal alternate support","tumor depth","tumor reference support","tumor alternate support"]}]},{"version":["latest"],"schema":[{"field":["chrom","pos_start","pos_end","n_snp","n_hom","n_het","filter"],"type":["c","i","i","i","i","i","c"],"description":["chromosome","start position","end position","SNP number","homozygous number","heterozygous number","filter"]}]},{"version":["latest"],"schema":[{"field":["qcstatus","contamination","consanguinity","uniparental_disomy"],"type":["c","d","d","c"],"description":["QC status","contamination percentage","consanguinity proportion","uniparental disomy"]}]},{"version":["latest"],"schema":[{"field":["genome_territory","mean_coverage","sd_coverage","median_coverage","mad_coverage","pct_exc_adapter","pct_exc_mapq","pct_exc_dupe","pct_exc_unpaired","pct_exc_baseq","pct_exc_overlap","pct_exc_capped","pct_exc_total","pct_1x","pct_5x","pct_10x","pct_15x","pct_20x","pct_25x","pct_30x","pct_40x","pct_50x","pct_60x","pct_70x","pct_80x","pct_90x","pct_100x"],"type":["i","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["","","","","","","","","","","","","","","","","","","","","","","","","","",null]}]},{"version":["latest"],"schema":[{"field":["coverage","count"],"type":["c","d"],"description":["Coverage","Count"]}]},{"version":["latest"],"schema":[{"field":["passed_or_failed","total","primary","secondary","suppl","dup","primary_dup","mapped","primary_map","paired_in_seq","read1","read2","proper_pair","both_map","singletons","matemap_diff","matemap_diff_mapq5","mapped_pct","primary_map_pct","proper_pair_pct","singletons_pct"],"type":["c","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["QC passed or failed","total number of reads that are QC pass and fail","neither secondary nor supplementary","secondary","supplementary","dup","dup and neither secondary nor supplementary","not unmap","not unmap, not secondary, and not supplementary","paired","both paired and read1","both paired and read2","both paired and proper_pair, not unmap","paired and neither unmap nor munmap","both paired and munmap, and not unmap","paired and neither unmap nor munmap, and mrnm not equal to rname","paired and neither unmap nor munmap, and mrnm not equal to rname, and mapq >= 5","percentage of total reads mapped","percentage of total primary reads","percentage against total pairs","percentage against total pairs"]}]},{"version":["latest"],"schema":[{"field":["length","count"],"type":["c","d"],"description":["fragment length","Count"]}]},{"version":["latest"],"schema":[{"field":["chromosome","pos_start","pos_end","total_reads","dup_reads","chim_reads","interpartition","unmap_reads","process_time"],"type":["c","d","d","d","d","d","d","d","d"]}]},{"version":["latest"],"schema":[{"field":["tot_region_bases","tot_reads","dup_reads","dual_strand_reads","cov_mean","cov_sd","cov_median","cov_mad","lowmapq_pct","dup_pct","unpaired_pct","lowbaseq_pct","overlap_read_pct","cov_capped"],"type":["d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["Total region bases","Total reads","Duplicate reads","Dual strand reads","Mean coverage","Standard deviation of coverage","Median coverage","Median absolute deviation of coverage","Low mapping quality percentage","Duplicate percentage","Unpaired percentage","Low base quality percentage","Overlapping read percentage","Coverage capped"]}]},{"version":["latest"],"schema":[{"field":["covx","value"],"type":["d","d"],"description":["Coverage x","Value"]}]},{"version":["latest"],"schema":[{"field":["sample_id","p_brca1","p_brca2","p_hrd","hr_status","hrd_type","remarks_hr_status","remarks_hrd_type"],"type":["c","d","d","d","c","c","c","c"],"description":["sample ID","probability of BRCA1-type HRD","probability of BRCA2-type HRD","probability of HRD","HR status (deficient or proficient)","HRD type (BRCA1 or BRCA2)","remarks on HR status","remarks on HRD type"]}]},{"version":["latest"],"schema":[{"field":["signature","count"],"type":["c","i"],"description":["signature context","count of signature"]}]},{"version":["latest"],"schema":[{"field":["gc_bucket","median"],"type":["c","d"],"description":["GC bucket number","median read depth in bucket"]}]},{"version":["latest"],"schema":[{"field":["chromosome","median_ratio","count"],"type":["c","d","d"],"description":["chromosome","median ratio","count"]}]},{"version":["latest"],"schema":[{"field":["sample_id","chrom","arm","start_pos","end_pos","n_probes","mean"],"type":["c","c","c","d","d","i","d"],"description":["sample ID","chromosome","chromosome arm","start position","end position","number of probes","mean"]}]},{"version":["latest"],"schema":[{"field":["version","date_build"],"type":["c","c"],"description":["version of the tool","build date"]}]},{"version":["latest"],"schema":[{"field":["category","result_type","data_type","value","ref_cancer_type","ref_value","plot_section"],"type":["c","c","c","c","c","d","c"],"description":["category","result type","data type","value","ref cancer type","ref value","section of cuppa plot"]}]},{"version":["latest"],"schema":[{"field":["source","category","key","value"],"type":["c","c","c","d"],"description":["Source","Category","Key","Value"]}]},{"version":["latest"],"schema":[{"field":["sample_id","clf_group","clf_name","pred_class_rank","pred_class","pred_prob_rank","pred_prob","extra_info","extra_info_format"],"type":["c","c","c","i","c","i","d","c","c"],"description":["sample ID","classifier group","classifier name","prediction class rank","prediction class","prediction probability rank","prediction probability","extra info","extra info format"]}]},{"version":["latest"],"schema":[{"field":["sample_id","data_type","clf_group","clf_name","feat_name","feat_value","cancer_type","data_value","rank","rank_group"],"type":["c","c","c","c","c","d","c","d","d","d"],"description":["sample ID","data type","classifier group","classifier name","EVENT feature name","EVENT feature value","cancer type","Value of data_type","rank of data_value","Grouping of [data_type, clf_name, feat_name] by which data_value is ranked"]}]},{"version":["latest"],"schema":[{"field":["fragment_length","count"],"type":["d","d"]}]},{"version":["latest"],"schema":[{"field":["tot_reads","written_reads","translocation","short_inv","del_dup_10k","del_dup_100k","del_dup_long","inv_100k","inv_long"],"type":["d","d","d","d","d","d","d","d","d"]}]},{"version":["latest"],"schema":[{"field":["chromosome","position","orientation","junc_frags","exact_support_frags","other_support_frags","low_map_qual_frags","max_qual","extra_info","indel","hotspot","initial_read_id"],"type":["c","d","d","d","d","d","d","d","d","c","c","c"]}]},{"version":["latest"],"schema":[{"field":["assembly_ids","assembly_info","merged","sequence_length","assembly_cigar","full_sequence"],"type":["c","c","c","d","c","c"]}]},{"version":["latest"],"schema":[{"field":["id","chromosome","junc_position","junc_orientation","junc_type","ext_base_length","ref_base_position","ref_base_length","ref_base_cigar","split_frags","ref_split_frags","disc_frags","ref_disc_frags","outcome","phase_group_id","phase_group_count","phase_set_id","phase_set_count","split_links","facing_links","sv_type","sv_length","inserted_bases","overlap_bases","ref_linked_frags","secondary_links","mismatch_reads","ref_base_trimmed","ref_base_trim_length","junc_sequence","ref_base_sequence","is_line","ref_base_candidates","unmapped_candidates","junc_supps","junc_indels","junc_mate_concord","junc_mate_ref_side","junc_mate_discord_remote","junc_mate_discord_ref_side","junc_mate_unmapped_remote","junc_mate_unmapped_ref_side","remote_region_count","remote_region_info","remote_region_junc_mate","remote_region_junc_supps","remote_region_discordant","assembly_info","initial_read_id","repeat_info","ref_side_soft_clips","merged_assemblies","soft_clip_matches","soft_clip_mismatches","soft_clip_2nd_max_length","ref_base_mismatches","base_trim_count","avg_nm_count","avg_indel_length","avg_base_qual","avg_map_qual","ext_base_build_info"],"type":["c","c","d","d","c","d","d","d","c","d","d","d","d","c","d","d","c","d","c","c","c","d","c","c","c","c","d","c","d","c","c","c","d","d","d","d","d","d","d","d","d","d","d","c","d","d","d","c","c","c","c","d","d","d","d","d","d","d","d","d","d","c"]}]},{"version":["latest"],"schema":[{"field":["id","phase_group_id","phase_set_id","assembly_id","mate_id","assembly_info","type","chromosome","position","orientation","mate_chr","mate_pos","mate_orient","length","inserted_bases","homology","confidence_interval","inexact_offset","qual","split_fragments","ref_split_fragments","disc_fragments","ref_disc_fragments","forward_reads","reverse_reads","sequence_length","segment_count","segment_index","sequence_index","aligned_bases","map_qual","score","adj_aligned_bases","avg_fragment_length","incomplete_fragments","breakend_qual","facing_breakend_ids","alt_alignments"],"type":["c","c","c","c","c","c","c","c","d","d","c","d","d","d","c","c","c","c","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","c","c"]}]},{"version":["latest"],"schema":[{"field":["assembly_ids","assembly_info","ref_info","raw_seq_coords","adj_seq_coords","map_qual","cigar","orientation","aligned_bases","score","flags","n_matches","xa_tag","md_tag","calc_align_length","mod_map_qual","dropped_on_requery","linked_alt_alignment"],"type":["c","c","c","c","c","d","c","d","d","d","d","d","c","c","d","d","c","c"]}]},{"version":["latest"],"schema":[{"field":["passed_or_failed","total","primary","secondary","suppl","dup","primary_dup","mapped","primary_map","paired_in_seq","read1","read2","proper_pair","both_map","singletons","matemap_diff","matemap_diff_mapq5","mapped_pct","primary_map_pct","proper_pair_pct","singletons_pct"],"type":["c","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["QC passed or failed","total number of reads that are QC pass and fail","neither secondary nor supplementary","secondary","supplementary","dup","dup and neither secondary nor supplementary","not unmap","not unmap, not secondary, and not supplementary","paired","both paired and read1","both paired and read2","both paired and proper_pair, not unmap","paired and neither unmap nor munmap","both paired and munmap, and not unmap","paired and neither unmap nor munmap, and mrnm not equal to rname","paired and neither unmap nor munmap, and mrnm not equal to rname, and mapq >= 5","percentage of total reads mapped","percentage of total primary reads","percentage against total pairs","percentage against total pairs"]}]},{"version":["latest"],"schema":[{"field":["sample_id","qc_status","frag_tot","frag_dup","frag_spliced_pct","frag_unspliced_pct","frag_alt_pct","frag_chimeric_pct","spliced_gene_count","read_length","frag_length_5th","frag_length_50th","frag_length_95th","enriched_gene_pct","median_gc_ratio","forward_strand_pct"],"type":["c","c","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["sample ID","QC status","total fragments","duplicate marked fragments","fragment % supporting 1 or more known transcripts","fragment % not containing a splice junction and not supporting any known transcript","fragment % supporting an alternate splice junction within a gene","fragment % supporting a chimeric junction","count of spliced genes","raw read length of fragments","5th percentile of genic intronic fragment lengths","50th percentile of genic intronic fragment lengths","95th percentile of genic intronic fragment lengths","fragment % supporting one of the following 6 genes: RN7SL2, RN7SL1, RN7SL3, RN7SL4P, RN7SL5P & RN7SK","median GC ratio excluding the 6 highly enriched genes",""]}]},{"version":["latest"],"schema":[{"field":["gene_id","gene_name","chromosome","gene_length","intronic_length","transcript_count","gene_set_id","spliced_fragments","unspliced_fragments","tpm_adj","tpm_raw","fit_residuals","low_map_quality_fragments"],"type":["c","c","c","i","i","i","c","i","i","d","d","d","d"],"description":["Ensembl gene id","Gene name","chromosome","length of gene","total bases which are intronic across all transcripts","count of transcripts in gene","identifier shared by all genes in overlapping group","count of fitted fragments assigned to spliced transcripts in the gene","count of fitted fragments assigned to the unspliced gene transcript","adjusted TPM","raw TPM","fit residuals",""]}]},{"version":["latest"],"schema":[{"field":["gene_id","gene_name","transcript_id","transcript_name","exon_count","transcript_length","effective_length","fitted_fragments","raw_fitted_fragments","tpm_adj","tpm_raw","transcript_bases_covered","sj_supported","sj_supported_unique","sj_fragments_unique","non_sj_fragments_unique","fragments_discordant","low_map_quality_fragments"],"type":["c","c","c","c","i","i","i","d","d","d","d","d","d","d","d","d","d","d"],"description":["Ensembl ID for gene","gene name","Ensembl ID for transcript","Ensembl transcript name","count of exons in transcript","total length of transcript","effective length of transcript (adjusted for fragment size)","count of fragments assigned to transcript","count of fragments assigned to transcript prior to bias adjustments","adjusted TPM","raw TPM","number of bases in transcript with at least 1 supporting fragment","count of splice junctions in transcript with at least 1 supporting fragment","count of unique splice junctions in transcript with at least 1 supporting fragment","count of fragments supporting a splice junction unique to the transcript","count of fragments uniquely supporting transcript but without a unique splice junction","",""]}]},{"version":["latest"],"schema":[{"field":["gene_id","gene_name","chromosome","sj_start","sj_end","type","frag_count","dp_start","dp_end","region_start","region_end","bases_start","bases_end","trans_start","trans_end","nearest_start_exon","nearest_end_exon","init_read_id"],"type":["c","c","c","d","d","c","d","d","d","c","c","c","c","c","c","d","d","c"],"description":["Ensembl ID for gene","gene name","chromosome","start position of novel splice junction","end position of novel splice junction","type of novel splice junction","fragment count","total depth at start_sj position","total depth at end_sj position","","","","","","","","",""]}]},{"version":["latest"],"schema":[{"field":["gene_id","gene_name","chromosome","strand","position","type","frag_count","spliced_frag_count","dp_total","transcript_info"],"type":["c","c","c","d","d","c","d","d","d","c"],"description":["Ensembl ID for gene","gene name","chromosome","strand","position of exon boundary with overlapping frags suggesting retained intron","donor or acceptor splice site","Count of all fragments which overlap splice site boundary","Count of fragments which overlap splice site boundary which contain another splice site (ie. evidence of being part of a spliced transcript)","depth at splice boundary","transcript ID and exon rank of all transcripts that match splice site boundary"]}]},{"version":["latest"],"schema":[{"field":["fusion_id","valid","chr_up","pos_up","orient_up","gene_id_up","gene_name_up","strand_up","junc_type_up","coverage_up","max_anchor_length_up","trans_data_up","chr_down","pos_down","orient_down","gene_id_down","gene_name_down","strand_down","junc_type_down","coverage_down","max_anchor_length_down","trans_data_down","sv_type","total_frags","split_frags","realigned_frags","discordant_frags","related_spliced_ids","cohort_count","read_type","init_read_id"],"type":["c","c","c","d","c","c","c","c","c","d","d","c","c","d","d","c","c","d","c","d","d","c","c","d","d","d","d","c","d","c","c"],"description":["fusion id","valid fusion","chromosome of up breakend","position of up breakend","orientation of up breakend","Ensembl ID of gene at up breakend","gene name at up breakend","strand of gene at up breakend","junction type at up breakend (known, canonical, unknown)","n of fragments overlapping the last base prior to the up breakend","maximum number of distinct matched bases by any 1 junction supporting fragment at the up breakend","transcript ids which contain a splice junction which includes matches the up breakend","chromosome of down breakend","position of down breakend","orientation of down breakend","Ensembl ID of gene at down breakend","gene name at down breakend","strand of gene at down breakend","junction type at down breakend (known, canonical, unknown)","n of fragments overlapping the last base prior to the down breakend","maximum number of distinct matched bases by any 1 junction supporting fragment at the down breakend","transcript ids which contain a splice junction which includes matches the down breakend","structural variant type","n fragments supporting the transcript = spliced + realigned + discordant","n fragments with supplementary or spliced alignments that match the chimeric junction","n fragments which overlap the breakpoint at either end by at least 3 bases and exactly match the reference sequence at the other breakend but are not split or spliced at the break junction by the aligner","","","","read type",""]}]},{"version":["latest"],"schema":[{"field":["fusion_id","valid","chr_up","pos_up","orient_up","gene_id_up","gene_name_up","strand_up","junc_type_up","coverage_up","max_anchor_length_up","chr_down","pos_down","orient_down","gene_id_down","gene_name_down","strand_down","junc_type_down","coverage_down","max_anchor_length_down","sv_type","total_frags","split_frags","realigned_frags","discordant_frags","related_spliced_ids","cohort_count","filter","known_fusion_type"],"type":["c","c","c","d","d","c","c","d","c","d","d","c","d","d","c","c","d","c","d","d","c","d","d","d","d","c","d","c","c"],"description":["fusion id","valid fusion","chromosome of up breakend","position of up breakend","orientation of up breakend","Ensembl ID of gene at up breakend","gene name at up breakend","strand of gene at up breakend","junction type at up breakend (known, canonical, unknown)","n of fragments overlapping the last base prior to the up breakend","maximum number of distinct matched bases by any 1 junction supporting fragment at the up breakend","chromosome of down breakend","position of down breakend","orientation of down breakend","Ensembl ID of gene at down breakend","gene name at down breakend","strand of gene at down breakend","junction type at down breakend (known, canonical, unknown)","n of fragments overlapping the last base prior to the down breakend","maximum number of distinct matched bases by any 1 junction supporting fragment at the down breakend","structural variant type","n fragments supporting the transcript = spliced + realigned + discordant","n fragments with supplementary or spliced alignments that match the chimeric junction","n fragments which overlap the breakpoint at either end by at least 3 bases and exactly match the reference sequence at the other breakend but are not split or spliced at the break junction by the aligner","","","","",""]}]},{"version":["latest"],"schema":[{"field":["gene_set_id","gene_count","chromosome","range_start","range_end","total_frags","duplicates","supporting_trans","unspliced","alt_sj","chimeric","low_map_qual","fwd_strand","rev_strand","genes"],"type":["c","d","c","d","d","d","d","d","d","d","d","d","d","d","c"],"description":["Gene set ID","Gene count","chromosome","","","","","","","","","","","",""]}]},{"version":["latest"],"schema":[{"field":["allele","ref_total","ref_unique","ref_shared","ref_wild","tumor_total","tumor_unique","tumor_shared","tumor_wild","rna_total","rna_unique","rna_shared","rna_wild","tumor_cn","somatic_missense","somatic_nonsense_or_frameshift","somatic_splice","somatic_synonymous","somatic_inframe_indel"],"type":["c","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["Allele ID","Total assigned fragments from reference BAM","Fragments uniquely assigned to allele","Fragments assigned to ref allele and others in this solution","Fragments matched to a wildcard allele","Total assigned fragments from tumor BAM","Fragments uniquely assigned to allele","Fragments assigned to allele and others in this solution","Fragments matched to a wildcard allele","Total assigned fragments from RNA BAM","Fragments uniquely assigned to allele","Fragments assigned to allele and others in this solution","Fragments matched to a wildcard allele","Copy number from Tumor/Ref fragment ratio and Purple copy number","Matched missense variants","Matched nonsense or frameshift variants","Matched splice variants","Matched synonymous variants","Matched inframe indels"]}]},{"version":["latest"],"schema":[{"field":["status","score_margin","next_solution_alleles","median_base_quality","hla_y_allele","discarded_indels","discarded_indel_max_frags","discarded_alignment_fragments","a_low_coverage_bases","b_low_coverage_bases","c_low_coverage_bases","a_types","b_types","c_types","total_fragments","fitted_fragments","unmatched_fragments","uninformative_fragments","hla_y_fragments","percent_unique","percent_shared","percent_wildcard","unused_amino_acids","unused_amino_acid_max_frags","unused_haplotypes","unused_haplotype_max_frags","somatic_variants_matched","somatic_variants_unmatched"],"type":["c","d","c","d","c","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["Either PASS or 1 or more warnings","Difference in score to second-top solution","Allele difference in second-top solution","Median base quality across all coding bases from all fragments","HLA-Y allele detected if present, else NONE","Discarded fragments due to unknown INDELs","Maximum fragment support for an indel detected but not present in any known allele","Fragments discarded because 1 read aligns more than 1000 bases from a HLA A,B or C gene","Number of bases with less than 15-depth coverage across all coding bases, also for B and C genes","Number of bases with less than 15-depth coverage across all coding bases, also for A and C genes","Number of bases with less than 15-depth coverage across all coding bases, also for A and B genes","# of distinct HLA-A alleles fitted (0,1 or 2)","# of distinct HLA-B alleles fitted (0,1 or 2)","# of distinct HLA-C alleles fitted (0,1 or 2)","Total of fragments overlapping a coding base of HLA-A, HLA-B or HLA-C with MAPQ >=1","Total fragments assigned to fitted alleles (ie. exact or wildcard match at every heterozygous location)","Fragment that do not match any allele exactly in solution at all heterozygous location (eg. due to sequencing error or mapping error)","Fragments that do not overlap any heterozygous location considered in evidence phase","Fragments excluded from fit due to allocation to assignment to HLA-Y","Percentage of fitted fragments that are uniquely assigned to 1 allele","Percentage of fitted fragments allocated across multiple alleles","Percentage of fitted fragments uniquely assigned to wildcard regions","# of amino acids detected with at least 3 fragments support but not present in final allele solution set (excluding PON filtered and regions overlapping wildcards in selected alleles)","Maximum fragments support for an unmatched amino acid not present in the final allele solution set","# of haplotypes phased with at least 3 fragments support but not present in final allele solution set (excluding PON filtered and regions overlapping wildcards in selected alleles)","Maximum support for an unmatched haplotype not present in final allele solution set","Somatic variants supported by solution allele","Somatic variants not supported by solution allele"]}]},{"version":["v1.25","latest"],"schema":[{"field":["bnd_id","sv_id","is_start","gene","transcript_id","canonical","gene_orientation","disruptive","reported_disruption","undisrupted_cn","region_type","coding_type","biotype","exonic_basephase","next_splice_exon_rank","next_splice_exon_phase","next_splice_distance","total_exon_count","type","chromosome","orientation","strand","chr_band","exon_up","exon_down","junction_cn"],"type":["c","c","c","c","c","c","c","c","c","d","c","c","c","d","d","d","d","d","c","c","c","c","c","d","d","d"],"description":["ID of breakend annotation","ID of break junction","annotation relates to the start breakend of the break junction","gene annotated","Ensembl stable transcript ID of annotation","transcript is the canonical transcript of the gene","orientation which breakend points relative to the gene taking into account both gene strand and breakend orientation","breakend is part of a break junction which disrupts the exonic sequence of the transcript","breakend is disruptive and gene is flagged as reportable for disruption","number of remaining wildtype alleles of the gene that are not disrupted by the breakend","Location of the breakend relative to the transcript","location of the breakend relative to the coding context of the transcript","Ensembl biotype of the transcript","if regionType = EXONIC, the exact base phase of the breakend, else -1","exon rank of the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")","phase of the 1st base after the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")","distance in bases to the next splice site identified in nextSpliceExonRank","total number of exons in the transcript","type of breakend","chromosome","orientation","strand","chromosome band",".",".","junction cn"]},{"field":["bnd_id","sv_id","is_start","gene","transcript_id","canonical","gene_orientation","disruptive","reported_disruption","undisrupted_cn","region_type","coding_type","biotype","exonic_basephase","next_splice_exon_rank","next_splice_exon_phase","next_splice_distance","total_exon_count","exon_up","exon_down"],"type":["c","c","c","c","c","c","c","c","c","d","c","c","c","d","d","d","d","d","d","d"],"description":["ID of breakend annotation","ID of break junction","annotation relates to the start breakend of the break junction","gene annotated","Ensembl stable transcript id of annotation","transcript is the canonical transcript of the gene","orientation which breakend points relative to the gene taking into account both gene strand and breakend orientation","breakend is part of a break junction which disrupts the exonic sequence of the transcript","breakend is disruptive and gene is flagged as reportable for disruption","number of remaining wildtype alleles of the gene that are not disrupted by the breakend","Location of the breakend relative to the transcript","location of the breakend relative to the coding context of the transcript","Ensembl biotype of the transcript","if regionType = EXONIC, the exact base phase of the breakend, else -1","exon rank of the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")","phase of the 1st base after the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")","distance in bases to the next splice site identified in nextSpliceExonRank","total number of exons in the transcript",".","."]}]},{"version":["latest"],"schema":[{"field":["cluster_id","category","synthetic","resolved_type","cluster_count","cluster_desc"],"type":["c","c","c","c","d","c"],"description":["Unique Id for the cluster","high level categorisation of the cluster classification","set to TRUE if the cluster is resolved to a non complex type by simplification of a short templated insertion (<1kb)","resolved classification of the cluster.","number of break junctions in the cluster","types and counts of break junctions in the cluster. eg. DEL=2_INV=2"]}]},{"version":["latest"],"schema":[{"field":["chromosome","chromosome_band","gene","transcript","is_canonical","driver","category","likelihood_method","driver_likelihood","missense","nonsense","splice","inframe","frameshift","biallelic","min_cn","max_cn"],"type":["c","c","c","c","c","c","c","c","d","d","d","d","d","d","c","d","d"],"description":["Chromosome of gene","Chromosome band of driver","Gene name","Transcript name","Is canonical","Driver type [AMP, DEL, MUTATION]","Gene driver type [ONCO, TSG]","Method used to determine likelihood","Likelihood that gene is a driver","Number of missense variants in gene","Number of nonsense variants in gene","Number of splice variants in gene","Number of inframe variants in gene","Number of frameshift variants in gene","True if any variants in the gene are biallelic","Minimum CN found in the gene exons","Maximum CN found in the gene exons"]}]},{"version":["latest"],"schema":[{"field":["cluster_id","gene","event_type"],"type":["c","c","c"],"description":["Id of cluster which break junction associated with driver","Gene of driver","Type of driver"]}]},{"version":["latest"],"schema":[{"field":["breakendid5","breakendid3","name","reported","reported_type","reportable_reasons","phased","likelihood","chain_length","chain_links","chain_terminated","domains_kept","domains_lost","skipped_exons_up","skipped_exons_down","fused_exon_up","fused_exon_down","gene_start","gene_context_start","transcript_start","gene_end","gene_context_end","transcript_end","junction_cn"],"type":["c","c","c","c","c","c","c","c","d","d","c","c","c","d","d","d","d","c","c","c","c","c","c","d"],"description":["5prime breakend ID in the fusion","3prime breakend ID in the fusion","fusion name in the form 5primeGENE_3primeGENE","fusion meets all reportable fusion criteria for LINX","if one or both of the genes matches a promiscuous gene or known rearrangement in the HMF fusion knowledgebase, then the type of reportable gene pair","reasons for given reportable status","set to 1 if a phased fusion can be formed (after allowing for exon skipping)","likelihood","0 for simple fusions. If fusion is chained equal to the total length of segments chained between 5prime and 3prime partners","0 for simple fusions. If fusion is chained equal to the number of segments chained between 5prime and 3prime partners","True if the fusion is interrupted either on the 5prime partner side by a chained breakend prior to the start of the 5prime gene or by a chained breakend prior to the last coding base of the 3prime gene","List of 3prime partner domains retained in fusion product (as annotated by PROSITE profiles)","List of 3prime partner domains lost in fusion product (as annotated by PROSITE profiles)","Count of splice donors required to be skipped on 5prime partner side to form an inframe fusion","Count of splice donors required to be skipped on 3prime partner side to form an inframe fusion","Last exon fused on 5prime partner side","First exon fused on 3prime partner side",".",".",".",".",".",".","."]}]},{"version":["latest"],"schema":[{"field":["cluster_id","chain_id","chain_index","chain_count","lower_sv_id","upper_sv_id","lower_breakend_is_start","upper_breakend_is_start","chromosome","arm","assembled","traversed_sv_count","length","junction_cn","junction_cn_uncertainty","pseudogene_info","ecdna"],"type":["c","c","c","d","c","c","c","c","c","c","c","d","d","d","d","c","c"],"description":["cluster ID which contains the link","chain ID to which the link belongs representing a multi-segment prediction of the derivative chromosome","position of the linked segment in the chain","total count of linked segments in the chan","svId of the leftmost breakend of the linked segment","svId of the rightmost breakend of the linked segment","lower breakend is the start breakend of the break junction","right breakend is the start breakend of the break junction","chromosome of the linked segment","arm (P/Q) of the linked segment","segment is linked by a GRIDSS assembly","number of other breakends that are located on the linked segment","length of the linked segment","predicted CN of the chain","uncertainty in the CN of the chain","if the segment precisely matches an exon of an ensembl gene, then contains details of the matching exon","link is predicted to be part of a DM / ecDNA chain"]}]},{"version":["latest"],"schema":[{"field":["gene_id_up","gene_name_up","chrom_up","pos_up","orientation_up","sv_id_up","gene_id_down","gene_name_down","chrom_down","pos_down","orientation_down","sv_id_down","junc_cn","cn","insert_seq","chain_length","transcripts_up","transcripts_down"],"type":["c","c","c","d","d","c","c","c","c","d","d","c","d","d","c","d","c","c"]}]},{"version":["latest"],"schema":[{"field":["vcf_id","sv_id","cluster_id","cluster_reason","fragile_site_start","fragile_site_end","is_foldback","linetype_start","linetype_end","junction_cn_min","junction_cn_max","gene_start","gene_end","local_topology_id_start","local_topology_id_end","local_topology_start","local_topology_end","local_ti_count_start","local_ti_count_end"],"type":["c","c","c","c","c","c","c","c","c","d","d","c","c","c","c","c","c","d","d"],"description":["ID of break junction for mapping to GRIDSS / PURPLE vcf","ID of break junction","ID of cluster which break junction is assigned to","Reason for clustering and svId of clustered break junction for other break junction(s) to which the variant has been clustered","Start breakend of break junction is in a known fragile site (T/F)","End breakend of break junction is in a known fragile site (T/F)","Break junction is classified as a foldback (T/F)","Start breakend of break junction is in a known or suspected line source region","End breakend of break junction is in a known or suspected line source region","Minimum bound JCN estimate for breakjunction","Maximum bound JCN estimate for breakjunction","Gene(s) overlapping start breakend of SV","Gene(s) overlapping end breakend of SV","ID for group of proximate breakends to the start breakend of break junction within an extending 5kb window","ID for group of proximate breakends to the end breakend of break junction within an extending 5kb window","Local breakend topology type at site of start breakend","Local breakend topology type at site of end breakend.","Number of chained templated insertions in local topology group of start breakend","Number of chained templated insertions in local topology group of end breakend"]}]},{"version":["v1.25","latest"],"schema":[{"field":["sample_id","chromosome","start","end","cn","baf"],"type":["c","c","d","d","d","d"],"description":["sample ID","","","","",""]},{"field":["chromosome","start","end","cn","baf"],"type":["c","d","d","d","d"],"description":["","","","",""]}]},{"version":["v1.25","latest"],"schema":[{"field":["sample_id","cluster_id","reportable","gene_name_up","transcript_up","chr_up","pos_up","strand_up","region_type_up","fused_exon_up","gene_name_down","transcript_down","chr_down","pos_down","strand_down","region_type_down","fused_exon_down"],"type":["c","c","c","c","c","c","c","c","c","c","c","c","c","c","c","c","c"],"description":["sample ID","","","","","","","","","","","","","","","",""]},{"field":["cluster_id","reportable","gene_name_up","transcript_up","chr_up","pos_up","strand_up","region_type_up","fused_exon_up","gene_name_down","transcript_down","chr_down","pos_down","strand_down","region_type_down","fused_exon_down"],"type":["c","c","c","c","c","c","c","c","c","c","c","c","c","c","c","c"],"description":["","","","","","","","","","","","","","","",""]}]},{"version":["v1.25","latest"],"schema":[{"field":["sample_id","cluster_id","gene","transcript","chromosome","annotation_type","exon_rank","exon_start","exon_end"],"type":["c","c","c","c","c","c","c","c","c"],"description":["sample ID","","","","","","","",""]},{"field":["cluster_id","gene","transcript","chromosome","annotation_type","exon_rank","exon_start","exon_end"],"type":["c","c","c","c","c","c","c","c"],"description":["","","","","","","",""]}]},{"version":["v1.25","latest"],"schema":[{"field":["sample_id","cluster_id","transcript","chromosome","start","end","info"],"type":["c","c","c","c","c","c","c"],"description":["sample ID","","","","","",""]},{"field":["cluster_id","transcript","chromosome","start","end","info"],"type":["c","c","c","c","c","c"],"description":["","","","","",""]}]},{"version":["v1.25","latest"],"schema":[{"field":["sample_id","cluster_id","chain_id","chromosome","pos_start","pos_end","link_ploidy","in_double_minute"],"type":["c","c","c","c","c","c","d","c"],"description":["sample ID","","","","","","",""]},{"field":["cluster_id","chain_id","chromosome","pos_start","pos_end","link_ploidy","in_double_minute"],"type":["c","c","c","c","c","d","c"],"description":["","","","","","",""]}]},{"version":["v1.25","latest"],"schema":[{"field":["sample_id","cluster_id","chain_id","sv_id","type","resolved_type","is_synthetic","chr_start","chr_end","pos_start","pos_end","orient_start","orient_end","info_start","info_end","junction_cn","in_double_minute"],"type":["c","c","c","c","c","c","c","c","c","d","d","d","d","c","c","d","c"],"description":["sample ID","","","","","","","","","","","","","","","",""]},{"field":["cluster_id","chain_id","sv_id","type","resolved_type","is_synthetic","chr_start","chr_end","pos_start","pos_end","orient_start","orient_end","info_start","info_end","junction_cn","in_double_minute"],"type":["c","c","c","c","c","c","c","c","d","d","d","d","c","c","d","c"],"description":["","","","","","","","","","","","","","","",""]}]},{"version":["latest"],"schema":[{"field":["version","build_date"],"type":["c","c"],"description":["version of the tool","build date"]}]},{"version":["latest"],"schema":[{"field":["chromosome","start","end","gene","min_cn","max_cn","somatic_regions","transcript_id","is_canonical","chromosome_band","min_regions","min_region_start","min_region_end","min_region_start_support","min_region_end_support","min_region_method","min_minor_allele_cn","depth_window_count"],"type":["c","d","d","c","d","d","d","c","c","c","d","d","d","c","c","c","d","d"],"description":["chromosome of gene","start location of gene transcript","end location of gene transcript","name of gene as per Ensembl and HGNC","minimum cn found in the gene exons","maximum cn found in the gene exons","count of somatic cn regions this gene spans","ensembl transcript ID","is canonical","chromosome band of the gene","number of somatic regions inside the gene that share the min cn","start base of the cn region overlapping the gene with the minimum cn","end base of the cn region overlapping the gene with the minimum cn","start support of the cn region overlapping the gene with the minimum cn","end support of the cn region overlapping the gene with the minimum cn","method used to determine cn of the cn region overlapping the gene with the minimum cn","minimum allele cn found over the gene exons - useful for identifying LOH events","count of cobalt windows covered by this segment"]}]},{"version":["latest"],"schema":[{"field":["chromosome","start","end","cn","baf_count","observed_baf","baf","segment_start_support","segment_end_support","method","depth_window_count","gc_content","min_start","max_start","minor_allele_cn","major_allele_cn"],"type":["c","d","d","d","d","d","d","c","c","c","d","d","d","d","d","d"],"description":["chromosome of cn segment","start base of cn segment","end base of cn segment","fitted absolute cn of segment adjusted for purity and ploidy","count of amber baf points covered by this segment","combined reference and tumor BAF in tumor sample unadjusted for purity and ploidy","tumor BAF after adjusted for purity and ploidy","type of structural variant support for the cn breakpoint at start of region","type of structural variant support for the cn breakpoint at end of region","method used to determine the cn of the region","count of cobalt windows covered by this segment","proportion of segment that is G or C","minimum start location of this segment if there is any uncertainty","maximum start location of this segment if there is any uncertainty","cn of minor allele adjusted for purity","cn of major allele adjusted for purity"]}]},{"version":["latest"],"schema":[{"field":["chromosome","chromosome_band","gene","transcript","is_canonical","driver","category","likelihood_method","driver_likelihood","missense","nonsense","splice","inframe","frameshift","biallelic","min_cn","max_cn"],"type":["c","c","c","c","c","c","c","c","d","d","d","d","d","d","c","d","d"],"description":["chromosome of the gene","chromosome band of driver","name of gene","ensembl transcript ID","is canonical (True/False)","driver type (amp, del, mutation)","gene driver type (oncogene or tumor suppressor)","method used to determine likelihood","likelihood that gene is a driver","number of missense variants in gene","number of nonsense variants in gene","number of splice variants in gene","number of inframe variants in gene","number of frameshift variants in gene","true if any variants in the gene are biallelic","minimum cn found in the gene exons","maximum cn found in the gene exons"]}]},{"version":["latest"],"schema":[{"field":["gene","chromosome","chromosome_band","region_start","region_end","depth_window_count","exon_start","exon_end","detection_method","germline_status","tumor_status","germline_cn","tumor_cn","filter","cohort_frequency","reported"],"type":["c","c","c","d","d","d","d","d","c","c","c","d","d","c","d","c"],"description":["name of gene as per Ensembl and HGNC","chromosome of gene","chromosome band of gene","start base of deletion","end base of deletion","count of cobalt windows covered by this segment","start base of deletion","end base of deletion","method used to determine the cn of the region","germline status of the deletion","tumor status of the deletion","germline cn of the deletion","tumor cn of the deletion","filter applied to the deletion","frequency of deletion in the HMF cohort","reported deletion"]}]},{"version":["latest"],"schema":[{"field":["purity","norm_factor","score","diploid_proportion","ploidy","somatic_penalty"],"type":["d","d","d","d","d","d"],"description":["purity of tumor in the sample","internal factor to convert tumor ratio to cn.","score of fit (lower is better)","proportion of cn regions that have 1 (+- 0.2) minor and major allele","average ploidy of the tumor sample after adjusting for purity","penalty from somatic variants with implied variant cn that are inconsistent with the minor and major allele cn"]}]},{"version":["latest"],"schema":[{"field":["purity","norm_factor","score","diploid_proportion","ploidy","gender","status","polyclonal_proportion","min_purity","max_purity","min_ploidy","max_ploidy","min_diploid_proportion","max_diploid_proportion","somatic_penalty","whole_genome_duplication","ms_indels_per_mb","ms_status","tml","tml_status","tmb_per_mb","tmb_status","sv_tumor_mutational_burden","run_mode","targeted"],"type":["d","d","d","d","d","c","c","d","d","d","d","d","d","d","d","c","d","c","d","c","d","c","d","c","c"],"description":["purity of tumor in the sample","internal factor to convert tumor ratio to cn.","score of fit (lower is better)","proportion of cn regions that have 1 (+- 0.2) minor and major allele","average ploidy of the tumor sample after adjusting for purity","one of male or female","either pass or one or more warning or fail status","proportion of copy number regions that are more than 0.25 from a whole cn","minimum purity with score within 10% of best","maximum purity with score within 10% of best","minimum ploidy with score within 10% of best","maximum ploidy with score within 10% of best","min estimation for diploid proportion","max estimation for diploid proportion","penalty from somatic variants with implied variant cn that are inconsistent with the minor and major allele cn","true if more than 10 autosomes have major allele cn > 1.5","microsatellite indels per mega base","microsatellite status (msi, mss or unknown if somatic variants not supplied)","tumor mutational load (# of missense variants in sample)","tumor mutational load status (high, low or unknown if somatic variants not supplied)","tumor mutational burden (#passing variants per Mb) per mega base","tumor mutational burden status (high, low or unknown if somatic variants not supplied)","total number of non inferred, non single passing structural variants detected in sample","tumor_germline, tumor or germline, set based on whether a tumor and/or reference sample were supplied","true if purple was run with target-region files"]}]},{"version":["latest"],"schema":[{"field":["qc_status","method","cn_segments","unsupported_cn_segments","purity","gender_amber","gender_cobalt","deleted_genes","contamination","germline_aberrations","mean_depth_amber","loh_percent"],"type":["c","c","i","i","d","c","c","i","d","c","d","d"],"description":["QC status","method used to determine the cn of the region (NORMAL, HIGHLY_DIPLOID, SOMATIC or NO_TUMOR)","total number of cn segments in sample","count of cn segments with no SV support on either side. A high number is an indicator of poor sample quality","purity of tumor in the sample","gender from amber data","gender from cobalt data","number of genes with cn < 0.5","rate of contamination in tumor sample as determined by Amber","any germline chromosomal aberrations detected (KLINEFELTER, TRISOMY_X, TRISOMY_21, TRISOMY_13, TRISOMY_18, TRISOMY_15, XYY, MOSAIC_X)","as calculated by Amber","LOH percentage"]}]},{"version":["latest"],"schema":[{"field":["peak","bucket","bucket_weight","peak_avg_weight","is_valid","is_subclonal"],"type":["d","d","d","d","c","c"],"description":["peak","bucket","bucket weight","peak average weight","is valid","is subclonal"]}]},{"version":["latest"],"schema":[{"field":["variant_cn_bucket","cn_bucket","count"],"type":["d","d","d"],"description":["variant cn bucket","cn bucket","count"]}]},{"version":["latest"],"schema":[{"field":["version","date_build"],"type":["c","c"],"description":["version of the tool","build date"]}]},{"version":["v3.4.4","latest"],"schema":[{"field":["alt","ref","context","count","origq","recalq"],"type":["c","c","c","d","d","d"],"description":["alternate allele","reference allele","trinucleotide context","count","original quality","recalibrated quality"]},{"field":["alt","ref","context","read_type","count","origq","recalq"],"type":["c","c","c","c","d","d","d"],"description":["alternate allele","reference allele","trinucleotide context","read type","count","original quality","recalibrated quality"]}]},{"version":["latest"],"schema":[{"field":["gene","chromosome","pos_start","pos_end","missed_var_likelihood","dr_0","dr_1","dr_2","dr_3","dr_4","dr_5","dr_6","dr_7","dr_8","dr_9","dr_10","dr_11","dr_12","dr_13","dr_14","dr_15","dr_16","dr_17","dr_18","dr_19","dr_20","dr_21","dr_22","dr_23","dr_24","dr_25","dr_26","dr_27","dr_28","dr_29","dr_30_39","dr_40_49","dr_50_59","dr_60_69","dr_70_79","dr_80_89","dr_90_99","dr_100_149","dr_150_199","dr_200_249","dr_250_299","dr_300_349","dr_350_399","dr_400_449","dr_450_499","dr_500_599","dr_600_699","dr_700_799","dr_800_899","dr_900_999","dr_1000_1099","dr_1100_1199","dr_1200_1299","dr_1300_1399","dr_1400_1499","dr_1500_1599","dr_1600_1699","dr_1700_1799","dr_1800_1899","dr_1900_1999","dr_2000_2999","dr_3000_3999","dr_4000_4999","dr_5000_5999","dr_6000_6999","dr_7000_7999","dr_8000_8999","dr_9000_9999","dr_10000plus"],"type":["c","c","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d","d"],"description":["gene name","chromosome","start position","end position","missed variant likelihood","depth range 0","depth range 1","depth range 2","depth range 3","depth range 4","depth range 5","depth range 6","depth range 7","depth range 8","depth range 9","depth range 10","depth range 11","depth range 12","depth range 13","depth range 14","depth range 15","depth range 16","depth range 17","depth range 18","depth range 19","depth range 20","depth range 21","depth range 22","depth range 23","depth range 24","depth range 25","depth range 26","depth range 27","depth range 28","depth range 29","depth range 30-39","depth range 40-49","depth range 50-59","depth range 60-69","depth range 70-79","depth range 80-89","depth range 90-99","depth range 100-149","depth range 150-199","depth range 200-249","depth range 250-299","depth range 300-349","depth range 350-399","depth range 400-449","depth range 450-499","depth range 500-599","depth range 600-699","depth range 700-799","depth range 800-899","depth range 900-999","depth range 1000-1099","depth range 1100-1199","depth range 1200-1299","depth range 1300-1399","depth range 1400-1499","depth range 1500-1599","depth range 1600-1699","depth range 1700-1799","depth range 1800-1899","depth range 1900-1999","depth range 2000-2999","depth range 3000-3999","depth range 4000-4999","depth range 5000-5999","depth range 6000-6999","depth range 7000-7999","depth range 8000-8999","depth range 9000-9999","depth range 10000+"]}]},{"version":["latest"],"schema":[{"field":["gene","chromosome","start","end","exon","dp_med"],"type":["c","c","d","d","d","d"],"description":["gene name","chromosome","start position","end position","exon rank","median depth"]}]},{"version":["latest"],"schema":[{"field":["signature","allocation","percent"],"type":["c","d","d"],"description":["Signature name","Signature allocation","Signature allocation percentage"]}]},{"version":["latest"],"schema":[{"field":["bucket_name","count"],"type":["c","i"],"description":["signature context bucket","SNV count in bucket"]}]},{"version":["latest"],"schema":[{"field":["taxid_genus","name_genus","reads_genus_tree","taxid_species","name_species","reads_species_tree","taxid_assigned","name_assigned","reads_assigned_tree","reads_assigned_direct","reference","reference_taxid","reference_kmer_count","alternate_kmer_count","rname","startpos","endpos","numreads","covbases","coverage","meandepth","meanbaseq","meanmapq","integrations","qc_status"],"type":["c","c","i","c","c","i","c","c","i","i","c","c","i","i","c","i","i","i","i","d","d","d","d","d","c"],"description":["NCBI taxonomy ID of genus of viral reference","NCBI taxonomy genus name","Number of reads assigned to any virus in that genus by kraken2","NCBI taxonomy ID of species of viral reference","NCBI taxonomy species name","Number of reads assigned to that species (and any sub-species) by kraken2","NCBI taxonomy ID of viral reference","NCBI taxonomy name of viral reference","Number of reads assigned to the viral reference or taxonomic descendents by kraken2","Number of reads assigned to the viral reference taxid by kraken2","name of viral reference used.","NCBI taxonomy ID of viral reference used. This can be a child taxid of taxid_assigned.","count of read kmers matching reference used","count of read kmers matching next best reference candidate","name of adjusted viral reference","start position of adjusted viral reference. Always 1","end position of adjusted viral reference. Always equal to the viral contig length","Number of reads mapped to adjusted viral reference","Number of bases with at least 1 read mapped","Percentage of viral sequence with at least one read mapped","Mean alignment depth","Mean base quality of bases mapped to adjusted viral reference","Mean mapping quality of reads mapped to adjusted viral reference","Number of integration breakpoints found.","QC status of viral integration"]}]},{"version":["v1.3","latest"],"schema":[{"field":["taxid","name","qc_status","integrations","interpretation","percentage_covered","mean_coverage","expected_clonal_coverage","reported","driver_likelihood"],"type":["c","c","c","d","c","d","d","c","c","c"],"description":["reference taxid of the virus that is called with VIRUSBreakend","name of the virus, matching with the taxid","QC status as reported by VIRUSBreakend","number of detected integrations of this virus into the sample genome as reported by VIRUSBreakend","output of the interpretation step of Virus Interpreter","percentage of the viral reference sequence that has been covered in the tumor sample as reported by VIRUSBreakend (\"coverage\")","average coverage of the viral genome as reported by VIRUSBreakend (\"meanDepth\")","expected coverage assuming the virus is clonally integrated once in the tumor DNA","boolean indicating whether the detected viral presence is considered a driver","driver likelihood of the virus as annotated in the reporting db"]},{"field":["taxid","name","qc_status","integrations","interpretation","percentage_covered","mean_coverage","expected_clonal_coverage","reported","blacklisted","driver_likelihood"],"type":["c","c","c","d","c","d","d","c","c","c","c"],"description":["reference taxid of the virus that is called with VIRUSBreakend","name of the virus, matching with the taxid","QC status as reported by VIRUSBreakend","number of detected integrations of this virus into the sample genome as reported by VIRUSBreakend","output of the interpretation step of Virus Interpreter","percentage of the viral reference sequence that has been covered in the tumor sample as reported by VIRUSBreakend (\"coverage\")","average coverage of the viral genome as reported by VIRUSBreakend (\"meanDepth\")","expected coverage assuming the virus is clonally integrated once in the tumor DNA","boolean indicating whether the detected viral presence is considered a driver","Virus should be blacklisted (e.g. in case of HIV)","driver likelihood of the virus as annotated in the reporting db"]}]}],"description":["Duplicate read count and frequency.","Picard duplication metrics.","BAF segments using PCF algorithm.","Entry at each homozygous site in the reference and tumor.","Regions of homozygosity found in the reference.","AMBER QC.","Mostly coverage metrics.","Mostly coverage metrics.","Counts of alignments for each FLAG type.","Fragment length counts.","Partition stats.","Coverage summary.","Coverage summary. (tbl2)","Predictions from the CHORD random forest.","Counts of SNV, INDEL and SV contexts.","GC median read depth.","Chromosome median ratios.","Segmented regions determined from the ratios (piecewise constant fit).","Version of the tool","Visualisation data (CUPPA v1).","Input features.","Prediction summary.","Visualisation data (CUPPA v2).","Fragment length stats.","Discordant read stats.","Candidate junctions.","Phased assemblies.","Breakend assemblies.","Breakends.","Realigned assemblies.","Counts of alignments for each FLAG type.","QC summary.","Gene level data.","Transcript level data.","Alternate splice junctions.","Retained introns.","All fusions.","Passed fusions.","Gene collection.","Lilac solution summary.","Lilac QC","Impact of each non PON filtered break junction on each overlapping gene","Clustering of all non PON filtered SV events and their resolved classification.","Reproduction of the driver catalog produced by PURPLE with homozygous disruptions events appended.","Linkage of drivers from driver catalog to SV cluster which contributed to those drivers including LOH, deletion, disruption and amplification events.","All inframe and out of frame fusions predicted in the sample including HMF fusion knowledgebase annotations.","Chromosomal segments joining break junction pairs predicted to be linked and phased in cis on the derivative chromosome.","Gene fusion neoepitopes.","Additional annotations of each non PON filtered breakjunction.","","","","","","","Version of the tool","Copy number alterations of each gene in the HMF gene panel.","Copy number profile of all (contiguous) segments of the tumor sample.","Significant amplifications and deletions that occur in the HMF gene panel.","Germline deletions.","Best fit per purity sorted by score.","Purity/ploidy fit summary.","QC metrics.","Clonality peak model data.","Somatic variant histogram data.","Version of the tool.","Base quality recalibration metrics.","Gene coverage metrics.","Exon coverage metrics.","Signature allocation.","SNV counts.","VIRUSBreakend analysis summary.","Processed oncoviral call/annotation data."],"row_id":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73]},"columns":[{"id":"n","name":"n","type":"numeric","maxWidth":70},{"id":"tool","name":"tool","type":"character"},{"id":"tbl","name":"tbl","type":["glue","character"]},{"id":"schema_version","name":"Schema","type":"list","cell":["<div><button id=\"btn_1_0\" onclick=\"toggleSchema(1, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_1_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">duplicate_read_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Duplicate read count<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frequency<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Frequency of duplicate reads<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_2_0\" onclick=\"toggleSchema(2, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_2_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">16 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">library<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">library on which the duplicate marking was performed<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unpaired_reads_examined<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of mapped reads examined which did not have a mapped mate pair<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read_pairs_examined<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of mapped read pairs examined<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">secondary_or_supplementary_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of reads that were either secondary or supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unmapped_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of unmapped reads examined<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unpaired_read_duplicates<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of fragments that were marked as duplicates<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read_pair_duplicates<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of read pairs that were marked as duplicates<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read_pair_optical_duplicates<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of read pair duplicates that were caused by optical duplication<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percent_duplication<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fraction of mapped sequence that is marked as duplicate<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">estimated_library_size<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">estimated number of unique molecules in the library based on PE duplication<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">SPLIT_HERE<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">internal use for splitting the schema.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bin<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage bin (maybe?)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cov_mult<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage multiple<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">all_sets<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">all sets<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">optical_sets<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">optical sets<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">non_optical_sets<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">non optical sets<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_3_0\" onclick=\"toggleSchema(3, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_3_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sampleid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chrom<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">arm<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome arm<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start_pos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end_pos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">n_probes<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of probes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mean<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">mean BAF value<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_4_0\" onclick=\"toggleSchema(4, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_4_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">10 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chrom<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of site<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reference allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">alt<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">alternate allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dp_normal<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">normal depth<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">refsup_normal<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">normal reference support<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">altsup_normal<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">normal alternate support<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dp_tumor<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor depth<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">refsup_tumor<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor reference support<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">altsup_tumor<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor alternate support<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_5_0\" onclick=\"toggleSchema(5, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_5_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chrom<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">n_snp<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">SNP number<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">n_hom<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">homozygous number<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">n_het<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">heterozygous number<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">filter<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">filter<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_6_0\" onclick=\"toggleSchema(6, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_6_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">4 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qcstatus<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC status<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">contamination<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">contamination percentage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">consanguinity<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">consanguinity proportion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">uniparental_disomy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">uniparental disomy<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_7_0\" onclick=\"toggleSchema(7, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_7_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">27 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">genome_territory<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mean_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sd_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">median_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mad_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_adapter<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_mapq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_dupe<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_unpaired<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_baseq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_overlap<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_capped<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_exc_total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_1x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_5x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_10x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_15x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_20x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_25x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_30x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_40x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_50x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_60x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_70x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_80x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_90x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pct_100x<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NA<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_8_0\" onclick=\"toggleSchema(8, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_8_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Coverage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Count<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_9_0\" onclick=\"toggleSchema(9, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_9_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">21 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">passed_or_failed<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC passed or failed<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of reads that are QC pass and fail<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">neither secondary nor supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">secondary<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">secondary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">suppl<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dup<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">dup<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary_dup<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">dup and neither secondary nor supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mapped<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">not unmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary_map<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">not unmap, not secondary, and not supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">paired_in_seq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read1<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and read1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read2<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and read2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">proper_pair<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and proper_pair, not unmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">both_map<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired and neither unmap nor munmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">singletons<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and munmap, and not unmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">matemap_diff<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired and neither unmap nor munmap, and mrnm not equal to rname<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">matemap_diff_mapq5<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired and neither unmap nor munmap, and mrnm not equal to rname, and mapq >= 5<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mapped_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage of total reads mapped<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary_map_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage of total primary reads<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">proper_pair_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage against total pairs<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">singletons_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage against total pairs<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_10_0\" onclick=\"toggleSchema(10, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_10_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment length<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Count<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_11_0\" onclick=\"toggleSchema(11, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_11_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">9 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dup_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chim_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">interpartition<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unmap_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">process_time<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_12_0\" onclick=\"toggleSchema(12, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_12_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">14 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tot_region_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total region bases<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tot_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total reads<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dup_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Duplicate reads<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dual_strand_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Dual strand reads<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cov_mean<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Mean coverage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cov_sd<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Standard deviation of coverage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cov_median<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Median coverage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cov_mad<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Median absolute deviation of coverage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">lowmapq_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Low mapping quality percentage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dup_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Duplicate percentage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unpaired_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Unpaired percentage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">lowbaseq_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Low base quality percentage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">overlap_read_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Overlapping read percentage<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cov_capped<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Coverage capped<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_13_0\" onclick=\"toggleSchema(13, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_13_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">covx<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Coverage x<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">value<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Value<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_14_0\" onclick=\"toggleSchema(14, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_14_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">8 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">p_brca1<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">probability of BRCA1-type HRD<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">p_brca2<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">probability of BRCA2-type HRD<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">p_hrd<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">probability of HRD<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">hr_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">HR status (deficient or proficient)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">hrd_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">HRD type (BRCA1 or BRCA2)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remarks_hr_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">remarks on HR status<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remarks_hrd_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">remarks on HRD type<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_15_0\" onclick=\"toggleSchema(15, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_15_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">signature<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">signature context<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of signature<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_16_0\" onclick=\"toggleSchema(16, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_16_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gc_bucket<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">GC bucket number<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">median<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">median read depth in bucket<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_17_0\" onclick=\"toggleSchema(17, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_17_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">3 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">median_ratio<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">median ratio<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_18_0\" onclick=\"toggleSchema(18, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_18_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chrom<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">arm<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome arm<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start_pos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end_pos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">n_probes<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of probes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mean<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">mean<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_19_0\" onclick=\"toggleSchema(19, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_19_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">version<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">version of the tool<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">date_build<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">build date<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_20_0\" onclick=\"toggleSchema(20, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_20_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">category<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">category<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">result_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">result type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">data_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">data type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">value<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">value<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_cancer_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ref cancer type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_value<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ref value<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">plot_section<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">section of cuppa plot<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_21_0\" onclick=\"toggleSchema(21, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_21_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">4 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">source<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Source<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">category<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Category<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">key<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Key<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">value<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Value<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_22_0\" onclick=\"toggleSchema(22, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_22_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">9 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">clf_group<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">classifier group<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">clf_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">classifier name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pred_class_rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">prediction class rank<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pred_class<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">prediction class<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pred_prob_rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">prediction probability rank<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pred_prob<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">prediction probability<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">extra_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">extra info<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">extra_info_format<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">extra info format<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_23_0\" onclick=\"toggleSchema(23, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_23_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">10 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">data_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">data type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">clf_group<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">classifier group<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">clf_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">classifier name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">feat_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">EVENT feature name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">feat_value<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">EVENT feature value<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cancer_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">cancer type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">data_value<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Value of data_type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">rank of data_value<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rank_group<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Grouping of [data_type, clf_name, feat_name] by which data_value is ranked<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_24_0\" onclick=\"toggleSchema(24, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_24_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_25_0\" onclick=\"toggleSchema(25, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_25_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">9 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tot_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">written_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">translocation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">short_inv<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">del_dup_10k<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">del_dup_100k<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">del_dup_long<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inv_100k<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inv_long<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_26_0\" onclick=\"toggleSchema(26, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_26_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">12 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">position<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exact_support_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">other_support_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">low_map_qual_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">extra_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">indel<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">hotspot<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">initial_read_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_27_0\" onclick=\"toggleSchema(27, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_27_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_ids<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">merged<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sequence_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_cigar<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">full_sequence<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_28_0\" onclick=\"toggleSchema(28, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_28_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">62 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_position<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ext_base_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_position<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_cigar<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">split_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_split_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">disc_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_disc_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">outcome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phase_group_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phase_group_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phase_set_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phase_set_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">split_links<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">facing_links<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inserted_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">overlap_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_linked_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">secondary_links<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mismatch_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_trimmed<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_trim_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_sequence<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_sequence<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_line<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_candidates<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unmapped_candidates<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_supps<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_indels<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_mate_concord<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_mate_ref_side<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_mate_discord_remote<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_mate_discord_ref_side<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_mate_unmapped_remote<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_mate_unmapped_ref_side<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remote_region_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remote_region_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remote_region_junc_mate<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remote_region_junc_supps<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">remote_region_discordant<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">initial_read_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">repeat_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_side_soft_clips<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">merged_assemblies<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">soft_clip_matches<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">soft_clip_mismatches<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">soft_clip_2nd_max_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_base_mismatches<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">base_trim_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">avg_nm_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">avg_indel_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">avg_base_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">avg_map_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ext_base_build_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_29_0\" onclick=\"toggleSchema(29, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_29_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">38 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phase_group_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phase_set_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mate_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">position<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mate_chr<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mate_pos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mate_orient<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inserted_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">homology<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">confidence_interval<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inexact_offset<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">split_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_split_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">disc_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_disc_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">forward_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reverse_reads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sequence_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">segment_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">segment_index<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sequence_index<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">aligned_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">map_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">score<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">adj_aligned_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">avg_fragment_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">incomplete_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">breakend_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">facing_breakend_ids<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">alt_alignments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_30_0\" onclick=\"toggleSchema(30, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_30_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">18 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_ids<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembly_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">raw_seq_coords<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">adj_seq_coords<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">map_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cigar<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">aligned_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">score<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">flags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">n_matches<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">xa_tag<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">md_tag<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">calc_align_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mod_map_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dropped_on_requery<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">linked_alt_alignment<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_31_0\" onclick=\"toggleSchema(31, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_31_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">21 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">passed_or_failed<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC passed or failed<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of reads that are QC pass and fail<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">neither secondary nor supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">secondary<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">secondary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">suppl<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dup<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">dup<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary_dup<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">dup and neither secondary nor supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mapped<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">not unmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary_map<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">not unmap, not secondary, and not supplementary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">paired_in_seq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read1<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and read1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read2<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and read2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">proper_pair<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and proper_pair, not unmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">both_map<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired and neither unmap nor munmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">singletons<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">both paired and munmap, and not unmap<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">matemap_diff<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired and neither unmap nor munmap, and mrnm not equal to rname<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">matemap_diff_mapq5<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">paired and neither unmap nor munmap, and mrnm not equal to rname, and mapq >= 5<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mapped_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage of total reads mapped<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">primary_map_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage of total primary reads<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">proper_pair_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage against total pairs<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">singletons_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage against total pairs<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_32_0\" onclick=\"toggleSchema(32, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_32_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">16 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qc_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC status<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_tot<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total fragments<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_dup<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">duplicate marked fragments<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_spliced_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment % supporting 1 or more known transcripts<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_unspliced_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment % not containing a splice junction and not supporting any known transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_alt_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment % supporting an alternate splice junction within a gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_chimeric_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment % supporting a chimeric junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">spliced_gene_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of spliced genes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">raw read length of fragments<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_length_5th<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">5th percentile of genic intronic fragment lengths<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_length_50th<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">50th percentile of genic intronic fragment lengths<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_length_95th<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">95th percentile of genic intronic fragment lengths<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">enriched_gene_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment % supporting one of the following 6 genes: RN7SL2, RN7SL1, RN7SL3, RN7SL4P, RN7SL5P & RN7SK<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">median_gc_ratio<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">median GC ratio excluding the 6 highly enriched genes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">forward_strand_pct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_33_0\" onclick=\"toggleSchema(33, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_33_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">13 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl gene id<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">length of gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">intronic_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total bases which are intronic across all transcripts<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of transcripts in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_set_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">identifier shared by all genes in overlapping group<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">spliced_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of fitted fragments assigned to spliced transcripts in the gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unspliced_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of fitted fragments assigned to the unspliced gene transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tpm_adj<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">adjusted TPM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tpm_raw<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">raw TPM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fit_residuals<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fit residuals<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">low_map_quality_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_34_0\" onclick=\"toggleSchema(34, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_34_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">18 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID for gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID for transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl transcript name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of exons in transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total length of transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">effective_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">effective length of transcript (adjusted for fragment size)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fitted_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of fragments assigned to transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">raw_fitted_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of fragments assigned to transcript prior to bias adjustments<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tpm_adj<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">adjusted TPM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tpm_raw<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">raw TPM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_bases_covered<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of bases in transcript with at least 1 supporting fragment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sj_supported<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of splice junctions in transcript with at least 1 supporting fragment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sj_supported_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of unique splice junctions in transcript with at least 1 supporting fragment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sj_fragments_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of fragments supporting a splice junction unique to the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">non_sj_fragments_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of fragments uniquely supporting transcript but without a unique splice junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fragments_discordant<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">low_map_quality_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_35_0\" onclick=\"toggleSchema(35, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_35_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">18 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID for gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sj_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position of novel splice junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sj_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position of novel splice junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">type of novel splice junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fragment count<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dp_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total depth at start_sj position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dp_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total depth at end_sj position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bases_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bases_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">trans_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">trans_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">nearest_start_exon<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">nearest_end_exon<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">init_read_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_36_0\" onclick=\"toggleSchema(36, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_36_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">10 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID for gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">strand<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">position<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of exon boundary with overlapping frags suggesting retained intron<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">donor or acceptor splice site<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frag_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Count of all fragments which overlap splice site boundary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">spliced_frag_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Count of fragments which overlap splice site boundary which contain another splice site (ie. evidence of being part of a spliced transcript)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dp_total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth at splice boundary<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript ID and exon rank of all transcripts that match splice site boundary<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_37_0\" onclick=\"toggleSchema(37, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_37_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">31 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fusion_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fusion id<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">valid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">valid fusion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation of up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID of gene at up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name at up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">strand of gene at up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_type_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">junction type at up breakend (known, canonical, unknown)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n of fragments overlapping the last base prior to the up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_anchor_length_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum number of distinct matched bases by any 1 junction supporting fragment at the up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">trans_data_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript ids which contain a splice junction which includes matches the up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation of down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID of gene at down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name at down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">strand of gene at down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_type_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">junction type at down breakend (known, canonical, unknown)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n of fragments overlapping the last base prior to the down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_anchor_length_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum number of distinct matched bases by any 1 junction supporting fragment at the down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">trans_data_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript ids which contain a splice junction which includes matches the down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">structural variant type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n fragments supporting the transcript = spliced + realigned + discordant<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">split_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n fragments with supplementary or spliced alignments that match the chimeric junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">realigned_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n fragments which overlap the breakpoint at either end by at least 3 bases and exactly match the reference sequence at the other breakend but are not split or spliced at the break junction by the aligner<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">discordant_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">related_spliced_ids<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cohort_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">read type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">init_read_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_38_0\" onclick=\"toggleSchema(38, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_38_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">29 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fusion_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fusion id<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">valid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">valid fusion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation of up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID of gene at up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name at up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">strand of gene at up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_type_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">junction type at up breakend (known, canonical, unknown)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n of fragments overlapping the last base prior to the up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_anchor_length_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum number of distinct matched bases by any 1 junction supporting fragment at the up breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation of down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl ID of gene at down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name at down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">strand of gene at down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_type_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">junction type at down breakend (known, canonical, unknown)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n of fragments overlapping the last base prior to the down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_anchor_length_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum number of distinct matched bases by any 1 junction supporting fragment at the down breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">structural variant type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n fragments supporting the transcript = spliced + realigned + discordant<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">split_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n fragments with supplementary or spliced alignments that match the chimeric junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">realigned_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">n fragments which overlap the breakpoint at either end by at least 3 bases and exactly match the reference sequence at the other breakend but are not split or spliced at the break junction by the aligner<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">discordant_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">related_spliced_ids<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cohort_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">filter<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">known_fusion_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_39_0\" onclick=\"toggleSchema(39, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_39_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">15 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_set_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene set ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene count<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">range_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">range_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">duplicates<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">supporting_trans<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unspliced<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">alt_sj<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chimeric<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">low_map_qual<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fwd_strand<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rev_strand<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">genes<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_40_0\" onclick=\"toggleSchema(40, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_40_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">19 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">allele<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Allele ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total assigned fragments from reference BAM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments uniquely assigned to allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_shared<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments assigned to ref allele and others in this solution<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref_wild<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments matched to a wildcard allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total assigned fragments from tumor BAM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments uniquely assigned to allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_shared<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments assigned to allele and others in this solution<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_wild<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments matched to a wildcard allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rna_total<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total assigned fragments from RNA BAM<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rna_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments uniquely assigned to allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rna_shared<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments assigned to allele and others in this solution<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rna_wild<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments matched to a wildcard allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Copy number from Tumor/Ref fragment ratio and Purple copy number<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_missense<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Matched missense variants<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_nonsense_or_frameshift<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Matched nonsense or frameshift variants<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_splice<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Matched splice variants<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_synonymous<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Matched synonymous variants<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_inframe_indel<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Matched inframe indels<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_41_0\" onclick=\"toggleSchema(41, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_41_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">28 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Either PASS or 1 or more warnings<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">score_margin<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Difference in score to second-top solution<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_solution_alleles<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Allele difference in second-top solution<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">median_base_quality<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Median base quality across all coding bases from all fragments<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">hla_y_allele<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">HLA-Y allele detected if present, else NONE<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">discarded_indels<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Discarded fragments due to unknown INDELs<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">discarded_indel_max_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Maximum fragment support for an indel detected but not present in any known allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">discarded_alignment_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments discarded because 1 read aligns more than 1000 bases from a HLA A,B or C gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">a_low_coverage_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of bases with less than 15-depth coverage across all coding bases, also for B and C genes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">b_low_coverage_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of bases with less than 15-depth coverage across all coding bases, also for A and C genes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">c_low_coverage_bases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of bases with less than 15-depth coverage across all coding bases, also for A and B genes<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">a_types<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"># of distinct HLA-A alleles fitted (0,1 or 2)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">b_types<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"># of distinct HLA-B alleles fitted (0,1 or 2)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">c_types<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"># of distinct HLA-C alleles fitted (0,1 or 2)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total of fragments overlapping a coding base of HLA-A, HLA-B or HLA-C with MAPQ >=1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fitted_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Total fragments assigned to fitted alleles (ie. exact or wildcard match at every heterozygous location)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unmatched_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragment that do not match any allele exactly in solution at all heterozygous location (eg. due to sequencing error or mapping error)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">uninformative_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments that do not overlap any heterozygous location considered in evidence phase<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">hla_y_fragments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Fragments excluded from fit due to allocation to assignment to HLA-Y<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percent_unique<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Percentage of fitted fragments that are uniquely assigned to 1 allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percent_shared<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Percentage of fitted fragments allocated across multiple alleles<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percent_wildcard<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Percentage of fitted fragments uniquely assigned to wildcard regions<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unused_amino_acids<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"># of amino acids detected with at least 3 fragments support but not present in final allele solution set (excluding PON filtered and regions overlapping wildcards in selected alleles)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unused_amino_acid_max_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Maximum fragments support for an unmatched amino acid not present in the final allele solution set<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unused_haplotypes<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"># of haplotypes phased with at least 3 fragments support but not present in final allele solution set (excluding PON filtered and regions overlapping wildcards in selected alleles)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unused_haplotype_max_frags<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Maximum support for an unmatched haplotype not present in final allele solution set<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_variants_matched<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Somatic variants supported by solution allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_variants_unmatched<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Somatic variants not supported by solution allele<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_42_0\" onclick=\"toggleSchema(42, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_42_1\" onclick=\"toggleSchema(42, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_42_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">26 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bnd_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of breakend annotation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">annotation relates to the start breakend of the break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene annotated<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl stable transcript ID of annotation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">canonical<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript is the canonical transcript of the gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation which breakend points relative to the gene taking into account both gene strand and breakend orientation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">disruptive<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">breakend is part of a break junction which disrupts the exonic sequence of the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported_disruption<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">breakend is disruptive and gene is flagged as reportable for disruption<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">undisrupted_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of remaining wildtype alleles of the gene that are not disrupted by the breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Location of the breakend relative to the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coding_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">location of the breakend relative to the coding context of the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">biotype<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl biotype of the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exonic_basephase<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">if regionType = EXONIC, the exact base phase of the breakend, else -1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_splice_exon_rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">exon rank of the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_splice_exon_phase<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">phase of the 1st base after the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_splice_distance<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">distance in bases to the next splice site identified in nextSpliceExonRank<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_exon_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of exons in the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">type of breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">strand<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_band<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome band<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">junction cn<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_42_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">20 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bnd_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of breakend annotation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">annotation relates to the start breakend of the break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene annotated<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl stable transcript id of annotation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">canonical<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript is the canonical transcript of the gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_orientation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation which breakend points relative to the gene taking into account both gene strand and breakend orientation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">disruptive<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">breakend is part of a break junction which disrupts the exonic sequence of the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported_disruption<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">breakend is disruptive and gene is flagged as reportable for disruption<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">undisrupted_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of remaining wildtype alleles of the gene that are not disrupted by the breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Location of the breakend relative to the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coding_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">location of the breakend relative to the coding context of the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">biotype<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Ensembl biotype of the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exonic_basephase<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">if regionType = EXONIC, the exact base phase of the breakend, else -1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_splice_exon_rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">exon rank of the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_splice_exon_phase<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">phase of the 1st base after the next splice acceptor (if gene orientation is \"DOWNSTREAM\") or splice donor (if gene orientation is \"UPSTREAM\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">next_splice_distance<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">distance in bases to the next splice site identified in nextSpliceExonRank<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">total_exon_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of exons in the transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_43_0\" onclick=\"toggleSchema(43, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_43_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Unique Id for the cluster<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">category<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">high level categorisation of the cluster classification<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">synthetic<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">set to TRUE if the cluster is resolved to a non complex type by simplification of a short templated insertion (<1kb)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">resolved_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">resolved classification of the cluster.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of break junctions in the cluster<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_desc<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">types and counts of break junctions in the cluster. eg. DEL=2_INV=2<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_44_0\" onclick=\"toggleSchema(44, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_44_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">17 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Chromosome of gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome_band<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Chromosome band of driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Transcript name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_canonical<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Is canonical<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">driver<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Driver type [AMP, DEL, MUTATION]<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">category<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene driver type [ONCO, TSG]<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">likelihood_method<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Method used to determine likelihood<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">driver_likelihood<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Likelihood that gene is a driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">missense<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of missense variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">nonsense<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of nonsense variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">splice<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of splice variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inframe<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of inframe variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frameshift<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of frameshift variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">biallelic<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">True if any variants in the gene are biallelic<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Minimum CN found in the gene exons<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Maximum CN found in the gene exons<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_45_0\" onclick=\"toggleSchema(45, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_45_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">3 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Id of cluster which break junction associated with driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene of driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">event_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Type of driver<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_46_0\" onclick=\"toggleSchema(46, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_46_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">24 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">breakendid5<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">5prime breakend ID in the fusion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">breakendid3<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">3prime breakend ID in the fusion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fusion name in the form 5primeGENE_3primeGENE<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fusion meets all reportable fusion criteria for LINX<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">if one or both of the genes matches a promiscuous gene or known rearrangement in the HMF fusion knowledgebase, then the type of reportable gene pair<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reportable_reasons<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reasons for given reportable status<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">phased<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">set to 1 if a phased fusion can be formed (after allowing for exon skipping)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">likelihood<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">likelihood<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">0 for simple fusions. If fusion is chained equal to the total length of segments chained between 5prime and 3prime partners<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_links<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">0 for simple fusions. If fusion is chained equal to the number of segments chained between 5prime and 3prime partners<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_terminated<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">True if the fusion is interrupted either on the 5prime partner side by a chained breakend prior to the start of the 5prime gene or by a chained breakend prior to the last coding base of the 3prime gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">domains_kept<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">List of 3prime partner domains retained in fusion product (as annotated by PROSITE profiles)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">domains_lost<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">List of 3prime partner domains lost in fusion product (as annotated by PROSITE profiles)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">skipped_exons_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Count of splice donors required to be skipped on 5prime partner side to form an inframe fusion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">skipped_exons_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Count of splice donors required to be skipped on 3prime partner side to form an inframe fusion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fused_exon_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Last exon fused on 5prime partner side<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fused_exon_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">First exon fused on 3prime partner side<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_context_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_context_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">.<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_47_0\" onclick=\"toggleSchema(47, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_47_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">17 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster ID which contains the link<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chain ID to which the link belongs representing a multi-segment prediction of the derivative chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_index<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">position of the linked segment in the chain<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total count of linked segments in the chan<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">lower_sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">svId of the leftmost breakend of the linked segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">upper_sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">svId of the rightmost breakend of the linked segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">lower_breakend_is_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">lower breakend is the start breakend of the break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">upper_breakend_is_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">right breakend is the start breakend of the break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of the linked segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">arm<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">arm (P/Q) of the linked segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">assembled<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">segment is linked by a GRIDSS assembly<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">traversed_sv_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of other breakends that are located on the linked segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">length of the linked segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">predicted CN of the chain<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn_uncertainty<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">uncertainty in the CN of the chain<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pseudogene_info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">if the segment precisely matches an exon of an ensembl gene, then contains details of the matching exon<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ecdna<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">link is predicted to be part of a DM / ecDNA chain<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_48_0\" onclick=\"toggleSchema(48, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_48_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">18 rows Ã— 2 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chrom_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_id_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chrom_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orientation_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junc_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">insert_seq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_length<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcripts_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcripts_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_49_0\" onclick=\"toggleSchema(49, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_49_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">19 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">vcf_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of break junction for mapping to GRIDSS / PURPLE vcf<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of break junction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID of cluster which break junction is assigned to<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_reason<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Reason for clustering and svId of clustered break junction for other break junction(s) to which the variant has been clustered<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fragile_site_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Start breakend of break junction is in a known fragile site (T/F)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fragile_site_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">End breakend of break junction is in a known fragile site (T/F)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_foldback<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Break junction is classified as a foldback (T/F)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">linetype_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Start breakend of break junction is in a known or suspected line source region<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">linetype_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">End breakend of break junction is in a known or suspected line source region<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn_min<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Minimum bound JCN estimate for breakjunction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn_max<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Maximum bound JCN estimate for breakjunction<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene(s) overlapping start breakend of SV<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Gene(s) overlapping end breakend of SV<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">local_topology_id_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID for group of proximate breakends to the start breakend of break junction within an extending 5kb window<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">local_topology_id_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ID for group of proximate breakends to the end breakend of break junction within an extending 5kb window<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">local_topology_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Local breakend topology type at site of start breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">local_topology_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Local breakend topology type at site of end breakend.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">local_ti_count_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of chained templated insertions in local topology group of start breakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">local_ti_count_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of chained templated insertions in local topology group of end breakend<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_50_0\" onclick=\"toggleSchema(50, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_50_1\" onclick=\"toggleSchema(50, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_50_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">baf<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_50_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">5 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">baf<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_51_0\" onclick=\"toggleSchema(51, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_51_1\" onclick=\"toggleSchema(51, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_51_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">17 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reportable<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_type_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fused_exon_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_type_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fused_exon_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_51_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">16 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reportable<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_type_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fused_exon_up<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene_name_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">strand_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_type_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">fused_exon_down<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_52_0\" onclick=\"toggleSchema(52, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_52_1\" onclick=\"toggleSchema(52, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_52_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">9 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">annotation_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_52_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">8 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">annotation_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_rank<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_53_0\" onclick=\"toggleSchema(53, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_53_1\" onclick=\"toggleSchema(53, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_53_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_53_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">info<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_54_0\" onclick=\"toggleSchema(54, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_54_1\" onclick=\"toggleSchema(54, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_54_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">8 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">link_ploidy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">in_double_minute<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_54_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">link_ploidy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">in_double_minute<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_55_0\" onclick=\"toggleSchema(55, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.25<\/button><button id=\"btn_55_1\" onclick=\"toggleSchema(55, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_55_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.25<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">17 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sample_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">sample ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">resolved_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_synthetic<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">info_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">info_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">in_double_minute<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_55_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">16 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cluster_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chain_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">resolved_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_synthetic<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chr_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">orient_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">info_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">info_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">junction_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">in_double_minute<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\"><\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_56_0\" onclick=\"toggleSchema(56, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_56_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">version<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">version of the tool<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">build_date<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">build date<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_57_0\" onclick=\"toggleSchema(57, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_57_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">18 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start location of gene transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end location of gene transcript<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of gene as per Ensembl and HGNC<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">minimum cn found in the gene exons<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum cn found in the gene exons<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_regions<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of somatic cn regions this gene spans<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript_id<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ensembl transcript ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_canonical<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">is canonical<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome_band<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome band of the gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_regions<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of somatic regions inside the gene that share the min cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_region_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start base of the cn region overlapping the gene with the minimum cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_region_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end base of the cn region overlapping the gene with the minimum cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_region_start_support<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start support of the cn region overlapping the gene with the minimum cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_region_end_support<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end support of the cn region overlapping the gene with the minimum cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_region_method<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">method used to determine cn of the cn region overlapping the gene with the minimum cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_minor_allele_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">minimum allele cn found over the gene exons - useful for identifying LOH events<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">depth_window_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of cobalt windows covered by this segment<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_58_0\" onclick=\"toggleSchema(58, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_58_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">16 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of cn segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start base of cn segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end base of cn segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">fitted absolute cn of segment adjusted for purity and ploidy<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">baf_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of amber baf points covered by this segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">observed_baf<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">combined reference and tumor BAF in tumor sample unadjusted for purity and ploidy<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">baf<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor BAF after adjusted for purity and ploidy<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">segment_start_support<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">type of structural variant support for the cn breakpoint at start of region<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">segment_end_support<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">type of structural variant support for the cn breakpoint at end of region<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">method<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">method used to determine the cn of the region<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">depth_window_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of cobalt windows covered by this segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gc_content<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">proportion of segment that is G or C<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">minimum start location of this segment if there is any uncertainty<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum start location of this segment if there is any uncertainty<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">minor_allele_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">cn of minor allele adjusted for purity<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">major_allele_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">cn of major allele adjusted for purity<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_59_0\" onclick=\"toggleSchema(59, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_59_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">17 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of the gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome_band<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome band of driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">transcript<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">ensembl transcript ID<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_canonical<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">is canonical (True/False)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">driver<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">driver type (amp, del, mutation)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">category<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene driver type (oncogene or tumor suppressor)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">likelihood_method<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">method used to determine likelihood<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">driver_likelihood<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">likelihood that gene is a driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">missense<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of missense variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">nonsense<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of nonsense variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">splice<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of splice variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">inframe<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of inframe variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">frameshift<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of frameshift variants in gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">biallelic<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">true if any variants in the gene are biallelic<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">minimum cn found in the gene exons<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum cn found in the gene exons<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_60_0\" onclick=\"toggleSchema(60, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_60_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">16 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of gene as per Ensembl and HGNC<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome of gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome_band<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome band of gene<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start base of deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">region_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end base of deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">depth_window_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of cobalt windows covered by this segment<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start base of deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end base of deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">detection_method<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">method used to determine the cn of the region<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">germline_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">germline status of the deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor status of the deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">germline_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">germline cn of the deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_cn<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor cn of the deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">filter<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">filter applied to the deletion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cohort_frequency<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">frequency of deletion in the HMF cohort<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reported deletion<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_61_0\" onclick=\"toggleSchema(61, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_61_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">purity<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">purity of tumor in the sample<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">norm_factor<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">internal factor to convert tumor ratio to cn.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">score<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">score of fit (lower is better)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">diploid_proportion<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">proportion of cn regions that have 1 (+- 0.2) minor and major allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ploidy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">average ploidy of the tumor sample after adjusting for purity<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_penalty<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">penalty from somatic variants with implied variant cn that are inconsistent with the minor and major allele cn<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_62_0\" onclick=\"toggleSchema(62, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_62_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">25 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">purity<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">purity of tumor in the sample<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">norm_factor<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">internal factor to convert tumor ratio to cn.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">score<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">score of fit (lower is better)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">diploid_proportion<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">proportion of cn regions that have 1 (+- 0.2) minor and major allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ploidy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">average ploidy of the tumor sample after adjusting for purity<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gender<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">one of male or female<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">either pass or one or more warning or fail status<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">polyclonal_proportion<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">proportion of copy number regions that are more than 0.25 from a whole cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_purity<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">minimum purity with score within 10% of best<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_purity<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum purity with score within 10% of best<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_ploidy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">minimum ploidy with score within 10% of best<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_ploidy<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">maximum ploidy with score within 10% of best<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">min_diploid_proportion<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">min estimation for diploid proportion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">max_diploid_proportion<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">max estimation for diploid proportion<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">somatic_penalty<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">penalty from somatic variants with implied variant cn that are inconsistent with the minor and major allele cn<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">whole_genome_duplication<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">true if more than 10 autosomes have major allele cn > 1.5<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ms_indels_per_mb<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">microsatellite indels per mega base<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ms_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">microsatellite status (msi, mss or unknown if somatic variants not supplied)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tml<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor mutational load (# of missense variants in sample)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tml_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor mutational load status (high, low or unknown if somatic variants not supplied)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tmb_per_mb<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor mutational burden (#passing variants per Mb) per mega base<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">tmb_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor mutational burden status (high, low or unknown if somatic variants not supplied)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">sv_tumor_mutational_burden<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of non inferred, non single passing structural variants detected in sample<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">run_mode<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">tumor_germline, tumor or germline, set based on whether a tumor and/or reference sample were supplied<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">targeted<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">true if purple was run with target-region files<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_63_0\" onclick=\"toggleSchema(63, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_63_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">12 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qc_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC status<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">method<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">method used to determine the cn of the region (NORMAL, HIGHLY_DIPLOID, SOMATIC or NO_TUMOR)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cn_segments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">total number of cn segments in sample<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">unsupported_cn_segments<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of cn segments with no SV support on either side. A high number is an indicator of poor sample quality<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">purity<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">purity of tumor in the sample<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gender_amber<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gender from amber data<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gender_cobalt<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gender from cobalt data<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">deleted_genes<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of genes with cn < 0.5<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">contamination<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">rate of contamination in tumor sample as determined by Amber<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">germline_aberrations<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">any germline chromosomal aberrations detected (KLINEFELTER, TRISOMY_X, TRISOMY_21, TRISOMY_13, TRISOMY_18, TRISOMY_15, XYY, MOSAIC_X)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mean_depth_amber<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">as calculated by Amber<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">loh_percent<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">LOH percentage<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_64_0\" onclick=\"toggleSchema(64, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_64_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">peak<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">peak<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bucket<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">bucket<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bucket_weight<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">bucket weight<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">peak_avg_weight<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">peak average weight<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_valid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">is valid<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">is_subclonal<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">is subclonal<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_65_0\" onclick=\"toggleSchema(65, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_65_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">3 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">variant_cn_bucket<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">variant cn bucket<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">cn_bucket<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">cn bucket<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_66_0\" onclick=\"toggleSchema(66, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_66_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">version<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">version of the tool<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">date_build<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">build date<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_67_0\" onclick=\"toggleSchema(67, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v3.4.4<\/button><button id=\"btn_67_1\" onclick=\"toggleSchema(67, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_67_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v3.4.4<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">alt<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">alternate allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reference allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">context<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">trinucleotide context<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">origq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">original quality<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">recalq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">recalibrated quality<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_67_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">7 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">alt<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">alternate allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">ref<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reference allele<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">context<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">trinucleotide context<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">read_type<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">read type<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">origq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">original quality<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">recalq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">recalibrated quality<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_68_0\" onclick=\"toggleSchema(68, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_68_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">74 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">pos_end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">missed_var_likelihood<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">missed variant likelihood<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_0<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 0<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_2<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_3<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 3<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_4<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 4<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_5<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 5<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_6<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 6<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_7<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 7<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_8<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 8<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_9<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 9<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_10<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 10<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_11<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 11<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_12<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 12<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_13<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 13<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_14<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 14<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_15<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 15<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_16<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 16<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_17<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 17<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_18<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 18<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_19<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 19<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_20<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 20<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_21<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 21<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_22<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 22<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_23<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 23<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_24<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 24<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_25<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 25<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_26<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 26<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_27<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 27<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_28<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 28<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_29<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 29<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_30_39<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 30-39<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_40_49<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 40-49<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_50_59<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 50-59<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_60_69<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 60-69<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_70_79<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 70-79<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_80_89<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 80-89<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_90_99<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 90-99<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_100_149<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 100-149<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_150_199<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 150-199<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_200_249<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 200-249<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_250_299<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 250-299<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_300_349<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 300-349<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_350_399<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 350-399<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_400_449<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 400-449<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_450_499<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 450-499<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_500_599<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 500-599<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_600_699<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 600-699<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_700_799<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 700-799<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_800_899<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 800-899<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_900_999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 900-999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1000_1099<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1000-1099<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1100_1199<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1100-1199<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1200_1299<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1200-1299<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1300_1399<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1300-1399<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1400_1499<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1400-1499<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1500_1599<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1500-1599<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1600_1699<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1600-1699<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1700_1799<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1700-1799<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1800_1899<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1800-1899<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_1900_1999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 1900-1999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_2000_2999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 2000-2999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_3000_3999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 3000-3999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_4000_4999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 4000-4999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_5000_5999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 5000-5999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_6000_6999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 6000-6999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_7000_7999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 7000-7999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_8000_8999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 8000-8999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_9000_9999<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 9000-9999<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dr_10000plus<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">depth range 10000+<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_69_0\" onclick=\"toggleSchema(69, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_69_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">6 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">gene<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">gene name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">chromosome<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">start<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">end<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">exon<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">exon rank<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">dp_med<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">median depth<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_70_0\" onclick=\"toggleSchema(70, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_70_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">3 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">signature<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Signature name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">allocation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Signature allocation<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percent<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Signature allocation percentage<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_71_0\" onclick=\"toggleSchema(71, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_71_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">2 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">bucket_name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">signature context bucket<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">SNV count in bucket<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_72_0\" onclick=\"toggleSchema(72, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_72_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">25 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">taxid_genus<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy ID of genus of viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">name_genus<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy genus name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reads_genus_tree<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of reads assigned to any virus in that genus by kraken2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">taxid_species<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy ID of species of viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">name_species<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy species name<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reads_species_tree<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of reads assigned to that species (and any sub-species) by kraken2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">taxid_assigned<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy ID of viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">name_assigned<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy name of viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reads_assigned_tree<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of reads assigned to the viral reference or taxonomic descendents by kraken2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reads_assigned_direct<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of reads assigned to the viral reference taxid by kraken2<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reference<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of viral reference used.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reference_taxid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">NCBI taxonomy ID of viral reference used. This can be a child taxid of taxid_assigned.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reference_kmer_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of read kmers matching reference used<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">alternate_kmer_count<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">count of read kmers matching next best reference candidate<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">rname<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of adjusted viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">startpos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">start position of adjusted viral reference. Always 1<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">endpos<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">end position of adjusted viral reference. Always equal to the viral contig length<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">numreads<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of reads mapped to adjusted viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">covbases<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">i<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of bases with at least 1 read mapped<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Percentage of viral sequence with at least one read mapped<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">meandepth<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Mean alignment depth<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">meanbaseq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Mean base quality of bases mapped to adjusted viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">meanmapq<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Mean mapping quality of reads mapped to adjusted viral reference<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">integrations<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Number of integration breakpoints found.<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qc_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC status of viral integration<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>","<div><button id=\"btn_73_0\" onclick=\"toggleSchema(73, 0)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">v1.3<\/button><button id=\"btn_73_1\" onclick=\"toggleSchema(73, 1)\" style=\"background-color: #e3f2fd; border: 1px solid #90caf9; border-radius: 16px; padding: 6px 14px; margin: 3px; font-size: 12px; cursor: pointer; color: #1565c0; font-weight: 500; transition: all 0.2s ease;user-select: none;\" onmouseover=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#bbdefb'\" onmouseout=\"if(this.style.backgroundColor !== 'rgb(21, 101, 192)') this.style.backgroundColor='#e3f2fd'\">latest<\/button><div id=\"schema_73_0\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: v1.3<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">10 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">taxid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reference taxid of the virus that is called with VIRUSBreakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of the virus, matching with the taxid<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qc_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC status as reported by VIRUSBreakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">integrations<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of detected integrations of this virus into the sample genome as reported by VIRUSBreakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">interpretation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">output of the interpretation step of Virus Interpreter<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage_covered<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage of the viral reference sequence that has been covered in the tumor sample as reported by VIRUSBreakend (\"coverage\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mean_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">average coverage of the viral genome as reported by VIRUSBreakend (\"meanDepth\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">expected_clonal_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">expected coverage assuming the virus is clonally integrated once in the tumor DNA<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">boolean indicating whether the detected viral presence is considered a driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">driver_likelihood<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">driver likelihood of the virus as annotated in the reporting db<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><div id=\"schema_73_1\" style=\"display: none;\"><div style=\"margin-top: 10px; padding: 10px; border: 1px solid #ddd; border-radius: 6px; background-color: #fafafa;\"><div style=\"font-weight: 600; margin-bottom: 8px; color: #333;\">Version: latest<\/div><div style=\"font-size: 12px; color: #666; margin-bottom: 10px;\">11 rows Ã— 3 columns<\/div><div style=\"overflow-x: auto; max-height: 300px;\"><table style=\"width: 100%; border-collapse: collapse; font-size: 12px;\"><thead><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">field<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">type<\/th><th style=\"border: 1px solid #ddd; padding: 6px; background-color: #f5f5f5; text-align: left;\">description<\/th><\/thead><tbody><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">taxid<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">reference taxid of the virus that is called with VIRUSBreakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">name<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">name of the virus, matching with the taxid<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">qc_status<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">QC status as reported by VIRUSBreakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">integrations<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">number of detected integrations of this virus into the sample genome as reported by VIRUSBreakend<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">interpretation<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">output of the interpretation step of Virus Interpreter<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage_covered<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">percentage of the viral reference sequence that has been covered in the tumor sample as reported by VIRUSBreakend (\"coverage\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">mean_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">d<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">average coverage of the viral genome as reported by VIRUSBreakend (\"meanDepth\")<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">expected_clonal_coverage<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">expected coverage assuming the virus is clonally integrated once in the tumor DNA<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">reported<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">boolean indicating whether the detected viral presence is considered a driver<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">blacklisted<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">Virus should be blacklisted (e.g. in case of HIV)<\/td><\/tr><tr><td style=\"border: 1px solid #ddd; padding: 6px;\">driver_likelihood<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">c<\/td><td style=\"border: 1px solid #ddd; padding: 6px;\">driver likelihood of the virus as annotated in the reporting db<\/td><\/tr><\/tbody><\/table><\/div><\/div><\/div><\/div>"],"html":true,"minWidth":130},{"id":"description","name":"description","type":["glue","character"]},{"id":"row_id","name":"row_id","type":"numeric","show":false}],"filterable":true,"searchable":true,"pagination":false,"highlight":true,"bordered":true,"striped":true,"theme":{"borderColor":"#dfe2e5","stripedColor":"#f8f9fa","highlightColor":"#f0f5ff","cellPadding":"12px 15px"},"dataKey":"d60a8b77082c7f4cf24ef197469aba8a"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>



<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'light-border',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config);
    }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script></main>
</div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by <a href="https://github.com/pdiakumis" class="external-link">Peter Diakumis</a>.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.3.</p>
</div>

    </footer>
</div>





  </body>
</html>
